From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Americus Maximus <americancodemax@gmail.com>
Date: Mon, 8 Jan 2024 11:51:38 -0500
Subject: [PATCH] 6092: Adding missing DirectX 7 IIDs.

Adding missing IIDs from DirectDraw, DirectInput, DirectPlay, DirectSound, DirectMusic, etc.
---
 .../Base/data/typeinfo/win32/msvcrt/iids.txt  | 338 +++++++++++++++++-
 1 file changed, 337 insertions(+), 1 deletion(-)

diff --git a/Ghidra/Features/Base/data/typeinfo/win32/msvcrt/iids.txt b/Ghidra/Features/Base/data/typeinfo/win32/msvcrt/iids.txt
index 825a04b1b..d2fbd0f44 100644
--- a/Ghidra/Features/Base/data/typeinfo/win32/msvcrt/iids.txt
+++ b/Ghidra/Features/Base/data/typeinfo/win32/msvcrt/iids.txt
@@ -376,6 +376,10 @@
 013E6D36-EF69-53BC-8DB8-C403BCA550ED IWindowsUpdateAdministratorStatics
 013E6E4C-E6A7-4FB5-B7FF-D9F5DA805C31 IWdsTransportNamespaceScheduledCastManualStart
 013F9F9C-B449-4EC7-A6D2-9D4F2FC70AE5 IDigitalCableTuningSpace
+01411840-7786-11CF-8F52-0040333594A3 D3DRMMaterialAmbientColor
+01411841-7786-11CF-8F52-0040333594A3 D3DRMMaterialDiffuseColor
+01411842-7786-11CF-8F52-0040333594A3 D3DRMMaterialSpecularColor
+01411843-7786-11CF-8F52-0040333594A3 D3DRMMaterialPower
 0141FDA4-8FC0-11CE-BD18-204C4F4F5020 ITransactionImportWhereabouts
 0141FDA5-8FC0-11CE-BD18-204C4F4F5020 ITransactionExport
 0144AD85-C04C-4CDD-871A-A0726D097CD4 IMapServiceStatics
@@ -392,6 +396,7 @@
 01776DF3-B9AF-4E50-9B1C-56E93116D704 EditionUpgradeHelper
 0178FAD1-B361-4B27-96AD-67C57EBF2E1D MenuItem
 019104BE-522A-5904-F52F-DE72010429E0 IApplication2
+0194C220-A303-11D0-9C4F-00A0C905425E IDirectPlayLobby2
 01954E6B-9254-4E6E-808C-C9E05D007696 IVssEnumMgmtObject
 019FFF21-ECE6-4FBC-BF40-8938D4813E27 ISemanticZoom
 01B266C7-FD96-4FF5-A3C5-D27A9BF44237 IAudioFrameInputNode
@@ -413,6 +418,7 @@
 01F0C067-966B-4130-B872-469E42BD4A7F ITextBoxAutomationPeerFactory
 01F3B398-9527-4736-94DB-5195878E97A8 IPMT
 01F5F85E-0A81-40DA-A7C8-21EF3AF8440C IWindowsMediaLibrarySharingServices
+01FD92E0-A2FF-11D0-9C4F-00A0C905425E DPAID_ModemW
 0207D327-7FB8-5566-8C34-F8342037B7F9 IPenDevice2
 0209194F-632D-5154-AB52-0B5D15B12505 IHolographicCamera6
 020A16E5-6A25-40A3-8C00-50A023F619D1 ISecondaryAuthenticationFactorAuthentication
@@ -425,6 +431,7 @@
 02382786-B16A-4FDB-BE3B-C4240E13AD0D IPhoneCallVideoCapabilities
 023BCC0C-DFAB-4A61-B074-490C6995580D IPrintWorkflowXpsReceiver2
 023F6F1F-83FE-449A-A822-C696442D16B0 IEqualizerEffectDefinition
+02538130-898F-11D0-9AD0-00A0C9A06E35 IDirectInputEffectDriver
 025D1823-6C7D-447B-BBDB-A3CBC3DFA2FC IImageBytes
 0261D114-246A-5296-BC85-590F53AA39C8 IMapTileBitmapRequestedEventArgs2
 02656950-2152-11D0-944C-00A0C905416E ITransactionCloner
@@ -552,6 +559,7 @@
 04C6895D-EAF2-4034-97F3-311DE9BE413A IUpdateSearcher3
 04C75BF8-3CE1-473B-ACC5-3CC4F5E94999 IWICImageEncoder
 04C7E980-9AC2-4768-B536-B68DA4957D02 IPhoneCallOrigin2
+04CB5622-32E5-11D2-AFA6-00AA0024D8B6 DMUS_PROP_WavesReverb
 04D967D3-A4F6-495C-9530-8579FCBA8AA9 IHttpContentRangeHeaderValue
 04DCF9EF-3406-5700-8FEC-77EBA4C5A74B IDisplayFence
 04DE169F-BA02-4850-8B55-D82D03BA6D7F ICoreInputViewTransferringXYFocusEventArgs
@@ -633,8 +641,9 @@
 064E24DD-29C2-4F83-9AC1-9EE11578BEB3 IJsonObject
 0655E396-25D0-11D3-9C26-00C04F8EF87C SpLexicon
 065CC3E9-0F88-448B-9B64-A95826C65B8F IMidiOutPortStatics
+06621075-E92E-11D1-A8C5-00C04FA3726E StandardMIDIFile
 06622D85-6856-4460-8DE1-A81921B41C4B OpenControlPanel
-06675A80-3B9B-11D2-B92F-00609797EA5B IDirectDrawSurface7 
+06675A80-3B9B-11D2-B92F-00609797EA5B IDirectDrawSurface7
 066ACBCA-8881-49C9-BB98-FAE16B4889E1 IPrinterScriptContext
 066B85A2-09B7-4446-8502-6FE6DCAA7309 IUsbDeviceStatics
 066FB2B7-33D1-4E7D-8367-CF81D0F79653 IBluetoothLEAdvertisement
@@ -700,6 +709,8 @@
 07CD4E9F-4B3C-42B8-84E7-895368718AF2 IHdmiDisplayMode2
 07D04B45-1B2F-4627-82FD-683F197E5A75 ITextBoxStatics5
 07D0AD06-6302-4D29-878B-19DA03FC228D IWindowPatternIdentifiersStatics
+07D43D03-6523-11D2-871D-00600893B1BD IDirectMusicPerformance
+07D916C0-E0AF-11CF-9C4E-00A0C905425E DPAID_ServiceProvider
 07DD0893-AA21-5E6F-A91B-11B2B3FD7BE3 IHolographicKeyboard
 07DDC146-FC3D-11D2-9D8C-00C04F72D980 ITuneRequest
 07DEEAA7-018F-4DDA-B399-FD4E13A31BEA IContainerContentChangingEventArgs
@@ -765,6 +776,7 @@
 08E80A98-B8E5-41C1-A3D8-D3ABFAE22675 ISmsMessageReceivedEventArgs
 08EC3E00-50B0-11CF-960C-0080C7F4EE85 FolderItemVerb
 08F17EF7-8E3F-4D9A-BE8A-108F9A012C67 IGuidanceNavigator
+08F2D8C9-37C2-11D2-B9F9-0000F875AC12 IDirectMusicPort
 08F327FF-85D5-48B9-AEE9-28511E339F9F IStorageFolderStatics
 08F3863A-0018-445B-93D2-646D1C5ED05C IAdvancedPhotoCaptureSettings
 08F4003F-BC71-49E1-874A-31DDFF9A27B9 IRemoteSystemSessionInvitationListener
@@ -853,9 +865,13 @@
 0A9CCDBC-D797-4563-9667-94EC5D79292D IMFRateSupport
 0AA1AE0A-FA0E-4B84-8644-E05FF8E5ACB5 IDXGIAdapter2
 0AA6ED32-6589-49DA-AFDE-594270CA0AAC ICameraIntrinsics
+0AA97844-C877-11D1-870C-00600893B1BD DMUS_PROP_SynthSink_DSOUND
+0AA97845-C877-11D1-870C-00600893B1BD DMUS_PROP_SynthSink_WAVE
 0AAA09C0-5014-447C-8809-AEE4CB0AF94A IConnectionProfileFilter3
 0AACF7A4-5E1D-49DF-8034-FB6A68BC5ED1 ICoreApplication
 0AB198EF-6477-4EE8-8812-6780EDB82D5E ITfThreadMgr2
+0AB1C530-4745-11D1-A7A1-0000F803ABFC IDirectPlay4
+0AB1C531-4745-11D1-A7A1-0000F803ABFC IDirectPlay4A
 0AB994FC-5B89-4C3C-B5E4-5282A5316C0A ITensorFloat16Bit
 0ABB48B8-9E01-49C9-A355-7D48CAF7D6CA IWebAccountProviderDeleteAccountOperation
 0ABC885F-E031-48E8-8590-70DE78363C6D IGpioChangeReader
@@ -1075,6 +1091,12 @@
 0DC5E6ED-3E16-4BF1-8F9A-A979878BC195 ITextProvider2
 0DCED981-636E-4A34-8A3F-8EE018639285 IPivotItemFactory
 0DE527A0-0C7E-11D2-A484-00C04F8EFB69 ICrSpiral
+0DE9EAA1-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMViewportInterpolator
+0DE9EAA2-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMFrameInterpolator
+0DE9EAA3-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMMeshInterpolator
+0DE9EAA6-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMLightInterpolator
+0DE9EAA7-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMMaterialInterpolator
+0DE9EAA8-3B84-11D0-9B6D-0000C0781BC3 CDirect3DRMTextureInterpolator
 0DED49D5-A8B7-4D5D-97A1-12B0C195874D IBDA_PinControl
 0DEE87DE-0547-11D2-A894-0000F8084F96 IPropFindMultiResponse
 0DF2C7E1-3435-11D0-81D0-00C04FD85AB4 ITransportCallback
@@ -1144,6 +1166,7 @@
 0F03F8FE-2B26-46F0-965A-212AA8D66B76 IThumbnailCachePrimer
 0F0CB9D7-E5F3-45E6-9063-FF7BBB193C47 IListPickerFlyout
 0F19F306-7036-494F-BC3C-780DF0CC524A ISplinePointKeyFrame
+0F1D6860-88D9-11CF-9C4E-00A0C905425E DPSPSERIAL
 0F1EF154-F65E-4536-8339-5944081DB2CA ICmsDetachedSignature
 0F2EF0BA-C8B9-4C36-8987-94D3DAFEDE18 IInkToolbarCustomPenButtonFactory
 0F2F3758-8EC7-4381-922B-8F6C07D288F3 IPathIOStatics
@@ -1238,6 +1261,8 @@
 10D43942-DD94-5998-B235-AFAFB6FA7935 IIppAttributeValueStatics
 10D7CC31-8B8F-4B9D-B22C-617103B55B08 IIsolatedWindowsEnvironmentOptions2
 10DC0586-132B-4CAC-8BB3-DB00AC8B7EE0 IVBSAXAttributes
+10DD46A8-775B-11CF-8F52-0040333594A3 D3DRMAnimationKey
+10DD46A9-775B-11CF-8F52-0040333594A3 D3DRMFloatKeys
 10DF43C8-1DBE-11D3-8B34-006097DF5BD4 IBrowserFrameOptions
 10E2414A-EC59-49D2-BC51-5ADD2C36FEBC IProvideRuntimeContext
 10EB6EC6-8138-48C1-BC65-E1F14207367C IResourceLoader2
@@ -1303,6 +1328,7 @@
 12009F94-0D14-5443-BC09-250311CE570B IPrintWorkflowPrinterJob
 1202DB60-1DAC-42C5-AED5-1ABDD432248E SyncMgrClient
 120AC970-9A75-4ACF-AAE7-09972BCF8794 IClaimedLineDisplay
+120F30C0-1629-11D0-941C-0080C80CFA7B IDirect3DRMTexture2
 1210CB94-5A92-4602-BE24-79F318AF4AF9 IAppxManifestDriverDependency
 1217D7A6-5039-418C-B042-9CBE256AFD6E ID3D11RasterizerState1
 1225E4DF-03A1-42F8-8B80-C9228CD9CF2E IAppBroadcastingStatus
@@ -1349,6 +1375,7 @@
 130B3C0A-F565-476E-ABD5-EA05AEE74C69 IHdmiDisplayInformation
 130C65D3-BDD3-11DC-A8A8-001321F1405F MBN_SMS_CONSTANTS
 130F5136-EC43-4F61-89C7-9801A36D2E95 IVisualTreeService2
+1318F560-912C-11D0-9DAA-00A0C90A43CB DPAID_TotalSize
 131EB0D3-D04E-47B1-837E-49405BF6F80F IBluetoothLEAdvertisementFilter
 1327A7C8-AE8A-4FB3-8150-136227C37E9A IWdsTransportTftpManager
 132E42AB-4E36-4DF6-B1C8-0CE62FD88B26 IXmlCharacterData
@@ -1356,9 +1383,24 @@
 133336BF-296A-463E-9FF9-01CD25691408 ITimedMetadataStreamDescriptor
 13372ADA-9753-4574-8966-93145A76C7F9 IPaymentShippingOption
 133ADCD4-19B4-4020-9FDC-842E78253B17 ISpPhoneticAlphabetConverter
+133EFE40-32DC-11D0-9CFB-00A0C90A43CB IDirectPlay3
+133EFE41-32DC-11D0-9CFB-00A0C90A43CB IDirectPlay3A
 133F47C6-12DC-4898-BD47-04967DE648BA IWebViewControlSite
 1347D106-CF3A-428A-A5CB-AC0D9A2A4338 IBDA_SignalStatistics
 134AC7FC-397A-403F-A6EC-1CE8BEDA15E5 IComboBoxItemAutomationPeerFactory
+1352A560-DA61-11CF-9B06-00A0C903A3B8 DDVPTYPE_BROOKTREE
+13541C20-8E33-11D0-9AD0-00A0C9A06E35 ConstantForce
+13541C21-8E33-11D0-9AD0-00A0C9A06E35 RampForce
+13541C22-8E33-11D0-9AD0-00A0C9A06E35 Square
+13541C23-8E33-11D0-9AD0-00A0C9A06E35 Sine
+13541C24-8E33-11D0-9AD0-00A0C9A06E35 Triangle
+13541C25-8E33-11D0-9AD0-00A0C9A06E35 SawtoothUp
+13541C26-8E33-11D0-9AD0-00A0C9A06E35 SawtoothDown
+13541C27-8E33-11D0-9AD0-00A0C9A06E35 Spring
+13541C28-8E33-11D0-9AD0-00A0C9A06E35 Damper
+13541C29-8E33-11D0-9AD0-00A0C9A06E35 Inertia
+13541C2A-8E33-11D0-9AD0-00A0C9A06E35 Friction
+13541C2B-8E33-11D0-9AD0-00A0C9A06E35 CustomForce
 135DA193-0961-4B6E-8766-F13B7FBCCD58 IPrintTaskOptionDetailsStatic
 135FF860-22B7-4DDF-B0F6-218F4F299A43 IWICStream
 13608090-E336-4C8F-9053-A92407CA7B1D ILampArrayBitmapEffectFactory
@@ -1501,6 +1543,8 @@
 162AC829-7115-43EC-B383-A7B71644069D ISelectorAutomationPeer
 162C6FB5-44D3-435B-903D-E613FA093FB5 IStorageProviderHandler
 1630852E-1263-465B-98E5-FE60FFEC4AC2 IXFeedEvents
+1630B820-7842-11CF-8F52-0040333594A3 D3DRMIndexedColor
+1630B821-7842-11CF-8F52-0040333594A3 D3DRMMeshVertexColors
 16369792-B03E-4BA1-8BB8-D28F4AB4D2C0 IErrorReportingSettings
 1637E0A7-829C-45BC-929B-A1E7EA78D89B IStorageLibraryContentChangedTrigger
 163BB1E0-6E00-11CF-837A-48DC04C10000 IHTMLLocation
@@ -1569,6 +1613,11 @@
 177F1AA9-151E-5EF7-B1D9-2FBA0B4B0D17 ISharedPackageContainer
 17890342-6760-4BB9-A58A-F7AA98B08C0E IAdaptiveMediaSource2
 178BEFF7-CC39-44A6-B951-4A5653F073FA IAudioTrackSupportInfo
+178F2F24-C364-11D1-A760-0000F875AC12 DMUS_PROP_GM_Hardware
+178F2F25-C364-11D1-A760-0000F875AC12 DMUS_PROP_GS_Hardware
+178F2F26-C364-11D1-A760-0000F875AC12 DMUS_PROP_XG_Hardware
+178F2F27-C364-11D1-A760-0000F875AC12 DMUS_PROP_DLS1
+178F2F28-C364-11D1-A760-0000F875AC12 DMUS_PROP_MemorySize
 1791587A-2A7C-4981-A2D8-AB99FFFC65B9 IControlProtected
 1791E8F6-21C7-4340-882A-A6A93E3FD73B ILaunchUIContext
 179D65E7-4658-4CB6-AA13-41D094EA255E IWindowActivatedEventArgs
@@ -1734,6 +1783,7 @@
 1A777EAA-47C8-4930-B2C9-8FEE1C1B0F3B IVMRMixerControl9
 1A78A0A5-937D-46D4-832B-94FF14DAB061 IFrameworkTemplateFactory
 1A79BE16-7F30-4E35-BEEF-9D2C7B6DCAE1 IAppointmentCalendarCancelMeetingRequestEventArgs
+1A82F8BC-3F8B-11D2-B774-0060083316C1 KSDATAFORMAT_SUBTYPE_DIRECTMUSIC
 1A8A9D02-E482-461B-B8C7-8EFAD1CCE590 IThreadPoolTimerStatics
 1A92B015-47C3-468A-9333-0C6517574883 IMagneticStripeReader
 1A953E49-B103-437E-9226-AB67971342F9 IWiFiDirectDeviceStatics2
@@ -1802,6 +1852,7 @@
 1BAF6522-5F64-42C5-8267-B9FE2215BFBD IGameController
 1BB12A62-2AD8-432B-8CCF-0C2C52AFCD5B IPackageExecutionStateChangeNotification
 1BB2384D-A825-40C2-A7F5-281EAEDF3B55 ITimedTextStyle
+1BB4AF80-A303-11D0-9C4F-00A0C905425E IDirectPlayLobby2A
 1BB617B8-3886-49DC-AF82-A6C90FA35DDA IFsrmMutableCollection
 1BC317FD-23CB-4610-B085-C8E1C87EBAA0 IImageFeatureValueStatics
 1BC3580E-7E4F-11D2-94F4-00C04F79F1D6 IMsgTrackLog
@@ -1893,6 +1944,7 @@
 1D1B5AE0-42D9-4021-8261-3987619512E9 IWSManEx2
 1D1D5AF7-0D53-4FA9-BD84-D5816A86F3FF IAudioNodeEmitterDecayModel
 1D24F9BD-BD19-5D95-B573-711F649FCBE9 IMenuBarItemFlyoutFactory
+1D262760-E957-11CF-A5D6-28DB04C10000 KSDATAFORMAT_SUBTYPE_MIDI
 1D2BCDB4-EF8B-5209-BFC9-C32116504803 IMiracastReceiverSession
 1D2EFD50-75CE-11D1-B75A-00A0C90564FE ISearchCommandExt
 1D37147F-97DF-4226-92CA-DDF453D7FD5A IToggleSwitchStatics
@@ -1934,6 +1986,7 @@
 1DA2015A-A8E4-4C1B-88C9-0B183ECC62E3 IMediaTransportControlsFactory
 1DA6292F-BC66-11CE-AAE3-00AA004C2737 IAuditControl
 1DAC3CDA-481F-5469-8470-82C4BA680A28 IDisplayAdapterStatics
+1DB1AE6B-E92E-11D1-A8C5-00C04FA3726E ConnectToDLSCollection
 1DB6ED6C-AA2C-5967-9035-56352DC69658 ISceneRendererComponentFactory
 1DB9A292-CC78-4173-BE45-B61E67283EA7 IGameBarStatics
 1DC7A369-B7A3-4DF2-9D61-EBA85A0343D2 IStorageLibraryChangeTrackerTriggerDetails
@@ -1942,6 +1995,7 @@
 1DCFAFE9-B45E-41B3-BB2A-561EB129AE98 IBDA_TransportStreamSelector
 1DD2DE1F-571B-44D8-8E80-A08A1578766E ICapturedFrame
 1DD9E8DA-1C77-4D40-B0CF-98FEFDFF9512 IDirect3D8
+1DE12AB1-C9F5-11CF-BFC7-444553540000 IDirectInputJoyConfig
 1DE1F3BE-A655-55AD-B2B6-EB240F880356 IAppWindowChangedEventArgs
 1DED80A7-53EA-424F-8A04-17FEA9ADC4F5 IBackgroundCopyGroup
 1DF0D7F1-B267-4D28-8B10-12E23202A5C4 IItemNameLimits
@@ -2271,6 +2325,8 @@
 24230452-FE54-40CC-94F3-FCC394C340D6 IMFMediaEngineTransferSource
 2424B3F2-EB23-40F1-91AA-74BDDEEA0883 IMFVideoProcessorControl3
 24264891-E80B-4FD3-B7CE-4FF2FAE8931F IEntity
+242F6BC1-3849-11D0-9B6D-0000C0781BC3 IDirect3DRMInterpolator
+242F6BC2-3849-11D0-9B6D-0000C0781BC3 IDirect3DRMObjectArray
 24329274-9EB7-11DC-AE98-F2B456D89593 ITsSbLoadBalancing
 2433BF8E-0F9B-435C-BA2C-180611978C30 ITfContextView
 24374CF1-CCEB-48BF-A514-41B0186F84C2 IXamlReader
@@ -2351,6 +2407,8 @@
 25C1659F-4497-4135-940F-EE96F4D6E934 IDurationHelper
 25C48C20-1C41-424F-8084-67C1CFE0A854 IAppointmentStore2
 25CBB996-92ED-457E-B28C-4774084BD562 GenericCredentialProvider
+25E609E0-B259-11CF-BFC7-444553540000 DirectInput
+25E609E1-B259-11CF-BFC7-444553540000 DirectInputDevice
 25E93A03-28EC-5889-569B-7E486B126F18 IChatMessageValidationResult
 25F700C8-D816-4057-A9DC-3CBDEE77E256 IUIAutomation5
 25F99FF7-9347-459A-9FAC-B2D0E11C1A0F IApplicationOverrides
@@ -2376,6 +2434,8 @@
 267C123D-6EA4-4C50-8B1D-50207AFF1E8A ILineStatics
 267DB0B3-55E3-4902-949B-DF8F5CEC0191 MSVidWebDVDToVideoRenderer
 26836C24-D89E-46F4-A29A-4F4D4F144155 IBackgroundDownloaderFactory
+268A0FA0-60F2-11D2-AFA6-00AA0024D8B6 DMUS_PROP_WriteLatency
+268A0FA1-60F2-11D2-AFA6-00AA0024D8B6 DMUS_PROP_WritePeriod
 268FE9C5-D187-43DF-BBBF-5BEC79BE3011 IDirect3D8sw
 269316D8-57BD-11D2-9EEE-00C04F797396 ISpeechVoice
 269AD56A-8A67-4129-BC8C-0506DCFE9880 ISWbemObjectEx
@@ -2387,6 +2447,7 @@
 26B59FAB-2092-5B95-B59E-F27F210689E7 IIconSourceElementFactory
 26BA702B-314A-4620-AAF6-7A51AA58FA18 IVideoFrameNative
 26C29808-6322-456A-AF82-80C8642F178B ILoggingActivity2
+26C66A70-B367-11CF-A024-00AA006157AC IDirectPlayLobbyA
 26CFBFF0-713C-4BEC-8803-E101F7B14ED3 ICompositionTarget
 26D709AC-F70B-4421-A96F-D2878FAFB00D IMachineGlobalObjectTable
 26D836A5-0C15-44C7-AC59-B0DA8728F240 IPTFilterLicenseRenewal
@@ -2534,6 +2595,7 @@
 28F54685-06FD-11D2-B27A-00A0C9223196 IKsControl
 28F5BC2C-8C23-4234-AD2E-FA5A3A426E9B INumeralSystemTranslator
 28F6D4C0-FB12-40A4-8167-7F4E03D2724C IHardwareToken
+28F97EF7-9538-11D2-97A9-00C04FA36E58 CommandParam2
 28FF92D3-8F80-42FB-944F-B2107BBD42E6 IWebAccountProviderUIReportOperation
 2901296F-C0B0-49D0-8E8D-AA940AF9C10B IStoreSendRequestResult2
 29033A2F-C923-5A77-BCAF-055FF415AD32 ITimelinePropertiesChangedEventArgs
@@ -2629,6 +2691,7 @@
 2A76E4B0-A896-4219-86B6-EA8CDCE98F56 IAdcController
 2A8E8CB1-1000-4416-8363-CC27FB877308 IInteractionTracker
 2A9093E4-35C4-4B1B-A791-0D99DB56DD1D IMediaStreamSourceStartingRequest
+2A91F713-A4BF-11D2-BBDF-00600833DBD8 DMUS_PROP_SamplePlaybackRate
 2A9DA250-C769-458C-AFE8-FEB8D4D3B2EF IBackgroundTransferBase
 2AA2B5FE-B846-4D07-810C-B21EE45320E3 NetProvisioning
 2AB14321-956C-46AD-9752-F87F55052FE3 IMidiControlChangeMessageFactory
@@ -2659,12 +2722,14 @@
 2B605031-2248-4B2F-ACF0-7CEE36FC5870 IBluetoothLEDevice4
 2B6AC0F0-7E03-11D0-A9E0-00AA00685C74 ISEORouter
 2B6B9446-B999-5535-9D69-53F092C780A1 IDisplayManagerStatics
+2B74F7C0-9154-11CF-A9CD-00AA006886E3 IDirectPlay2
 2B7E40AA-DE07-424F-83F1-F1DE46C4FA2E IMediaBinder
 2B87245C-ED80-4FE5-8EB4-708A39C841E5 IDockPatternIdentifiersStatics
 2B89EFEC-1BEA-451E-8BCD-CFE2D9423262 IMediaTransportControls2
 2B8DE2FE-8D2D-11D1-B2FC-00C04FD915A9 IXMLDocument2
 2B8DE2FF-8D2D-11D1-B2FC-00C04FD915A9 IXMLElement2
 2B90F17E-5A3E-4B33-BB7F-1BC48056B94D ISyncMgrSyncResult
+2B957100-9E9A-11CF-AB39-0020AF71E433 D3DRMInfo
 2B998C3A-555C-4889-8ED8-C599BB3A712A IUnifiedPosErrorData
 2B9AE9EC-55EF-43B6-B459-12E40FFA4302 IPopupStatics2
 2BA042D6-373E-5A3A-A2FC-A6C41BD52789 ITensorUInt8BitStatics2
@@ -2674,12 +2739,14 @@
 2BB52CAF-F24A-46D0-B463-65F719731057 IMapControlBusinessLandmarkPointerExitedEventArgs
 2BC095A8-8EF0-446D-9E60-3A3E0428C671 ICoreApplicationView5
 2BC468E3-E1F8-4B54-95E8-32919551CE62 IBitmapEncoder
+2BC49361-8327-11CF-AC4A-0000C03825A1 IDirect3DRM
 2BCBA359-1CEF-40B2-9ECB-F927F856AE3C IPosPrinterReleaseDeviceRequestedEventArgs
 2BCE4ECE-D30E-445A-9423-6829BE945AD8 IImgErrorInfo
 2BCFCBD4-2657-4128-AD5F-E3877132BDB1 ISmsMessageReceivedTriggerDetails
 2BD8A6D0-2FA3-4717-B28C-4917CE54928D ICustomNavigationProvider
 2BD8F33A-2974-4759-A5A5-58F44D3C0275 IEmailMailboxForwardMeetingRequestEventArgs
 2BE49737-7C13-4C44-86C3-8B25E697A80E INavigationView2
+2BEFC277-5497-11D2-BCCB-00A0C922E6EB IDirectMusicGraph
 2BF1464E-FC71-47ED-85A1-3BA9577718B4 IBounceEase
 2BF755C5-E799-41B4-BB40-242F40959B71 IHoldingEventArgs
 2C012EC3-D975-47FB-96AC-A6418B326D2B IMediaRenderer
@@ -2791,6 +2858,8 @@
 2D9BD838-7C60-4842-9170-346FE10A226A IDragUI
 2DA3DB4E-E583-5055-AFAA-16FD986531A6 IManipulationStartedEventArgs2
 2DAFA41A-3EF8-4BB5-A02B-3EE1B2274740 ISelectionItemPatternIdentifiers
+2DB72490-652C-11D1-A7A8-0000F803ABFC IDirectPlayLobby3
+2DB72491-652C-11D1-A7A8-0000F803ABFC IDirectPlayLobby3A
 2DBE63C4-B340-48A0-A5B0-158E07FC567E IFsrmActionReport
 2DBEAD57-50A6-499E-8C6C-D330A7311796 IGameBarServices
 2DD06BA1-CE1A-44E6-84F9-3A92EA1E3044 IPrintQualityOptionDetails
@@ -2895,6 +2964,8 @@
 2FD364A6-BF36-42F7-B7E7-923C0AABFE16 IChatQueryOptions
 2FDF87E5-028D-491E-A058-3382C3986F40 ISmartCardConnect
 2FE352EA-FD1F-11D2-B1F4-00C04F8EEB3E ShellFolderItem
+2FE8F810-B2A5-11D0-A787-0000F803ABFC DirectPlayLobby
+2FFAACA2-5DCA-11D2-AFA6-00AA0024D8B6 IDirectMusicLoader
 3002D9A0-CFBF-4C0B-8F3C-66F503C20B39 IVpnCustomEditBox
 300E5D8A-1FF3-4D2C-95EC-27F81DEBACB8 IPropertyPath
 301035B5-2D42-4F56-8C65-2DCAA7FB3CDC IAntimalware2
@@ -3955,6 +4026,7 @@
 332C4425-26CB-11D0-B483-00C04FD90119 IHTMLDocument2
 332C4426-26CB-11D0-B483-00C04FD90119 IHTMLFramesCollection2
 332C4427-26CB-11D0-B483-00C04FD90119 IHTMLWindow2
+332CF160-DA61-11CF-9B06-00A0C903A3B8 DDVPTYPE_PHILIPS
 332E5848-2E15-458E-85C4-C911C0C3D6F4 IRandomAccessStreamFileAccessMode
 332EC562-3758-468D-A784-E38F23552128 IMFExtendedDRMTypeSupport
 33323223-6247-5419-A5A4-3D808645D917 IPanelBasedOptimizationControl
@@ -4105,6 +4177,8 @@
 35F1A0D0-3E9A-11D2-8499-005345000000 OfflineFolderStatus
 35F9CFC1-4176-44F4-B756-E9CCC4399805 ITextBox5
 35FE1BB8-A3A9-40FE-BBEC-EB569C9CCCA3 IMFWorkQueueServices
+35FF44E0-6C7C-11CF-8F52-0040333594A3 D3DRMColorRGBA
+35FF44E1-6C7C-11CF-8F52-0040333594A3 D3DRMMaterialArray
 36027933-A18E-4DB5-A697-F58FA7364E44 IRoutePolicyFactory
 360DEFB9-A9D3-4984-A4ED-9EC734604921 IPickerReturnedActivatedEventArgs
 36116642-D713-4B97-9B83-7484A9D00433 IFileDialogControlEvents
@@ -4157,6 +4231,7 @@
 36DBBA28-9E2E-49C6-8EF7-3AB7A5DCC8B8 IAppointmentsProviderLaunchActionVerbsStatics
 36DC7AEF-33E6-4691-AFE1-2BE7274B3D33 IRangeValueProvider
 36E0C96E-B56A-4B3B-8BAC-7AEF5E6F9945 IGridViewHeaderItemFactory
+36E95EE0-8577-11CF-960C-0080C7534E82 DPSPTCPIP
 36ED64E5-7266-4545-8E14-1645EEBA2A90 GPMEnum
 36F23359-040E-48F7-9A98-F2664591959C IStackPanel2
 36F26D09-C7D7-434F-B461-979DDFC2322F ICompass2
@@ -4343,6 +4418,8 @@
 3A0BEE95-29E4-51BF-8095-A3C068E3C72A IAppUriHandlerHost2
 3A14545D-801A-40BD-A286-AFB2010CCE6C IGuidanceRoute
 3A1E0B12-32DA-4C17-BFD7-8D0C73C8F5A5 IZoomControl
+3A23EEA0-94B1-11D0-AB39-0020AF71E433 D3DRMInlineData
+3A23EEA1-94B1-11D0-AB39-0020AF71E433 D3DRMUrl
 3A30FA01-5C40-499D-B33F-A43050F74FC4 IAppointmentManagerStatics
 3A31AC7D-28BF-4E7A-8524-71679D480F38 IRenderingDeviceReplacedEventArgs
 3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8 IDataTransferManagerInterop
@@ -4448,6 +4525,7 @@
 3C2654C6-7372-4F6B-B310-55D6128F49D2 AlphabeticalCategorizer
 3C2A477A-5CD9-3525-BA2A-23D1E0A68A1D IOcrWord
 3C2FAC6C-A977-47FC-B44E-2754C0B2BEA9 ISemanticZoomAutomationPeer
+3C305196-50DB-11D3-9CFE-00C04FD930C5 DirectDraw7
 3C36668A-5F18-4F0B-9CE5-CAB61D196F11 IAppExtensionCatalogStatics
 3C374A41-BAE4-11CF-BF7D-00AA006946EE IUrlHistoryStg
 3C374A42-BAE4-11CF-BF7D-00AA006946EE IEnumSTATURL
@@ -4539,6 +4617,20 @@
 3D7C353C-0D04-45F1-A742-F97CC1188DC8 IDVBSLocator
 3D7E4A33-3BD0-4902-8AC8-470D50D21376 ISymmetricKeyAlgorithmProvider
 3D813DFE-6C91-4A4E-8F41-04346A841D9C MXXMLWriter30
+3D82AB40-62DA-11CF-AB39-0020AF71E433 IDirectXFile
+3D82AB41-62DA-11CF-AB39-0020AF71E433 IDirectXFileEnumObject
+3D82AB42-62DA-11CF-AB39-0020AF71E433 IDirectXFileSaveObject
+3D82AB43-62DA-11CF-AB39-0020AF71E433 IDirectXFileObject
+3D82AB44-62DA-11CF-AB39-0020AF71E433 IDirectXFileData
+3D82AB45-62DA-11CF-AB39-0020AF71E433 IDirectXFileDataReference
+3D82AB46-62DA-11CF-AB39-0020AF71E433 IDirectXFileBinary
+3D82AB4A-62DA-11CF-AB39-0020AF71E433 D3DRMLight
+3D82AB4D-62DA-11CF-AB39-0020AF71E433 D3DRMMaterial
+3D82AB4F-62DA-11CF-AB39-0020AF71E433 D3DRMAnimation
+3D82AB50-62DA-11CF-AB39-0020AF71E433 D3DRMAnimationSet
+3D82AB51-62DA-11CF-AB39-0020AF71E433 D3DRMCamera
+3D82AB5E-62DA-11CF-AB39-0020AF71E433 D3DRMVector
+3D82AB5F-62DA-11CF-AB39-0020AF71E433 D3DRMMeshFace
 3D84945F-037B-404F-9BBB-6232D69543C3 IBarometerReadingChangedEventArgs
 3D8B0590-F691-11D2-8EA9-006097DF5BD4 IDataObjectAsyncCapability
 3D8F82E3-6CC6-423E-8D7C-7014D954DDEF ICalendarViewFactory
@@ -4908,6 +5000,7 @@
 44DFE30B-C3BE-40F5-BF82-7A95BB795ADF ITsSbPluginNotifySink
 44E6C007-75FB-4ABB-9F8B-824748900C71 ILockScreenUnlockingEventArgs
 44E82707-08CA-434A-AFE0-D61580446F7E IGpioProvider
+44EAA760-CB68-11CF-9C4E-00A0C905425E DPSPMODEM
 44EC053B-400F-11D0-9DCD-00A0C90391D3 IRegistrar
 44EC9B57-DCAB-4CDE-B373-EE94962950E3 INTERNETEXPLORERCONFIGURATION
 44FA4A24-2D09-42E1-8B5E-368208796A4C IEdgeGestureEventArgs
@@ -4920,6 +5013,18 @@
 45101AB7-7C3A-4BCE-9500-12C09024B298 ICastingSourceInfo
 45130DC1-BC61-44D4-A423-54712AE671C4 IInkPresenterRuler2
 4516E002-18E0-4079-8B5F-D33495C15D2E IMediaBreakFactory
+4516EC40-8F20-11D0-9B6D-0000C0781BC3 CDirect3DRMProgressiveMesh
+4516EC41-8F20-11D0-9B6D-0000C0781BC3 CDirect3DRM
+4516EC43-8F20-11D0-9B6D-0000C0781BC3 CDirectXFile
+4516EC77-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMMeshBuilder2
+4516EC78-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMDevice2
+4516EC79-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMProgressiveMesh
+4516EC7B-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMPicked2Array
+4516EC7C-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMObject2
+4516EC81-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMFace2
+4516EC82-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRMMeshBuilder3
+4516EC83-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRM3
+4516ECC8-8F20-11D0-9B6D-0000C0781BC3 IDirect3DRM2
 451FE266-DA6D-406A-BB60-82E534F85AEB IVdsLunControllerPorts
 45206A4C-0A4E-4FBF-AEE9-335D5A205F6E ITextBlock2
 45226402-95FC-4847-93B3-134DBF5C60F7 IPrint3DWorkflowPrinterChangedEventArgs
@@ -4961,6 +5066,10 @@
 45F406A7-FC9F-4F81-ACAD-355FEF51E67B ISocketActivityTriggerDetails
 45F4F6F3-D602-4F98-9A8A-3EFA152AD2D3 IInstallationCompletedCallback
 45F6D0BD-1773-492E-9BA3-8AC1CBE77C08 ISpatialInteractionSourceState2
+45FC707B-1DB4-11D2-BCAC-00A0C922E6EB DisableTimeSig
+45FC707C-1DB4-11D2-BCAC-00A0C922E6EB EnableTimeSig
+45FC707D-1DB4-11D2-BCAC-00A0C922E6EB DisableTempo
+45FC707E-1DB4-11D2-BCAC-00A0C922E6EB EnableTempo
 4600C3A5-E301-41D8-B6D0-EF2E4212E0CA Property
 46024401-2B2A-4F3C-A53C-1A045A408EFA IInkToolbarCustomPenButton
 46027FC0-D71B-45C7-8F14-6D9A0AE604E4 IVideoMediaFrameFormat
@@ -5057,6 +5166,7 @@
 47C01F95-E185-412C-B5C5-4F27DF965AEA IFolderBandPriv
 47C460EE-D5AA-445F-B31E-50C076C011B9 ISwipeItemsFactory
 47C922A2-3DD5-11D2-BF8B-00C04FB93661 ISearches
+47D4D946-62E8-11CF-93BC-444553540000 DirectSound
 47D579B7-0985-5E99-B024-2F32C380C1A4 IInteractionTrackerCustomAnimationStateEnteredArgs2
 47D6BA7B-E909-453F-866E-5540CDA734FF ICreateAudioFileOutputNodeResult
 47DABC31-10D3-4518-86F1-09CAF742D113 IBounceVector3NaturalMotionAnimation
@@ -5067,6 +5177,7 @@
 480BF94A-09FD-4F8A-A3E0-B0700282D84D IAdapterInfo
 480C7011-57AA-4DB6-B6FD-EC676F6D414E ISettingsFlyoutFactory
 480D41D5-7F99-474C-86EE-96DF168318F1 IVpnTrafficFilterFactory
+480FF4B0-28B2-11D1-BEF7-00C04FBF8FEF DirectMusicCollection
 4817D721-85EB-470C-8F37-43CA5498E41D IMediaFrameSourceInfo4
 4818EDAA-04E1-4127-9A4C-19351B8A80A4 IGeovisitTrigger
 48201E67-E08E-42A4-B561-41D08CA9575D IContact3
@@ -5089,8 +5200,11 @@
 486943E8-D187-4A6B-A3C4-291259FFF60D IDiaEnumDebugStreamData
 486B24E8-A94C-4090-8F48-09F782A9F7D5 ILightSensor2
 4876D0FF-9F4C-5F72-ADBD-A3471B063C00 ILightSensor3
+48784000-D219-11D0-BA39-00C04FD7ED67 DPLPROPERTY_PlayerScore
 487E84F6-94E2-4DCE-8833-1A700A37A29A ICertificateRequestProperties
 4881D0F2-ABC5-4FC6-84B0-89370BB47212 IAppBroadcastPlugInStateChangedEventArgs
+4885AE60-78E8-11CF-8F52-0040333594A3 D3DRMMaterialWrap
+4885AE63-78E8-11CF-8F52-0040333594A3 D3DRMBoolean2d
 48907C9C-DCD8-4C33-9AD3-6CDCE7B1C566 ITimedTextSourceResolveResultEventArgs
 48934BCF-CF2C-4EFC-9331-75BD22DF1F0C IAudioNodeEmitterNaturalDecayModelProperties
 489B0CAF-9EC2-4EB7-91F5-B6F71D43DA8C GPMRSOP
@@ -5177,6 +5291,7 @@
 4A055B70-B9AE-4458-9241-A6A5FBF18A0C IMobileBroadbandDeviceServiceTriggerDetails
 4A073526-6103-4E21-B7BC-F519D1524E5D IGetServiceIds
 4A1560D1-6992-4D9D-8555-4CA4563FB166 IPrintTaskSourceRequestedDeferral
+4A1B1BE6-BFED-11D1-8ED8-00A0C967A482 IDirect3DRMViewport2
 4A1DCBCB-26E9-4EB4-8933-859EB6ECDB29 IStartScreenManager
 4A249B72-FC9A-11D1-8B1E-00600806D9B6 WbemChangeFlagEnum
 4A249B73-FC9A-11D1-8B1E-00600806D9B6 WbemFlagEnum
@@ -5329,6 +5444,9 @@
 4D239005-3C2A-41B1-9022-536BB9CF93B1 ICoreWindowStatic
 4D25ACCB-797D-4FD2-B128-DFFEFF84FCC3 IMILBitmapEffectRenderContextImpl
 4D2FCB0A-7366-4971-8BD5-17C4E3E8C6C0 IPrint3DManager
+4D36E96B-E325-11CE-BFC1-08002BE10318 KeyboardClass
+4D36E96C-E325-11CE-BFC1-08002BE10318 MediaClass
+4D36E96F-E325-11CE-BFC1-08002BE10318 MouseClass
 4D3C06F9-C8DF-41CC-8BD5-2A947B204503 IWebViewControl2
 4D469C2E-3B75-4A73-BB30-736645472256 IContactManagerForUser2
 4D46F2C9-28BA-4970-8C7B-1F0C9D80AF69 IMFCameraControlMonitor
@@ -5459,6 +5577,20 @@
 4F93749B-DAC3-4A42-9CBB-99F0DE37C071 IRichTextBlockOverflow
 4F9F9FCB-E0F4-48EB-B7AB-FA2EA9365CB4 IHttpNegotiate2
 4FA1B53F-3C34-4868-A407-FC5EAB259B21 IPrintCustomItemListOptionDetails3
+4FA3568E-623F-11CF-AC4A-0000C03825A1 CDirect3DRMDevice
+4FA3568F-623F-11CF-AC4A-0000C03825A1 CDirect3DRMViewport
+4FA35690-623F-11CF-AC4A-0000C03825A1 CDirect3DRMFrame
+4FA35691-623F-11CF-AC4A-0000C03825A1 CDirect3DRMMesh
+4FA35692-623F-11CF-AC4A-0000C03825A1 CDirect3DRMMeshBuilder
+4FA35693-623F-11CF-AC4A-0000C03825A1 CDirect3DRMFace
+4FA35694-623F-11CF-AC4A-0000C03825A1 CDirect3DRMLight
+4FA35695-623F-11CF-AC4A-0000C03825A1 CDirect3DRMTexture
+4FA35696-623F-11CF-AC4A-0000C03825A1 CDirect3DRMWrap
+4FA35697-623F-11CF-AC4A-0000C03825A1 CDirect3DRMMaterial
+4FA35698-623F-11CF-AC4A-0000C03825A1 CDirect3DRMAnimation
+4FA35699-623F-11CF-AC4A-0000C03825A1 CDirect3DRMAnimationSet
+4FA3569A-623F-11CF-AC4A-0000C03825A1 CDirect3DRMUserVisual
+4FA3569B-623F-11CF-AC4A-0000C03825A1 CDirect3DRMShadow
 4FA6AABB-3F4B-4301-BE07-1172EA61EEFB ICheckBoxFactory
 4FAA3FFF-6746-4824-A964-EED8E87F8709 IConnectivityInterval
 4FAC1D43-FC51-45BB-B1B4-2B53AA562FA3 IRDPSRAPIInvitation
@@ -5717,6 +5849,7 @@
 536CED14-EBCD-49FE-BF9F-40EFC689B20D IRfcommDeviceService2
 53730A09-F968-4692-A177-C074597186DB IPrintTaskOptionDetails2
 537CD71E-0E46-4173-9001-BA043F3E49E2 IISDB_BIT
+537DA6A0-CA37-11D0-941C-0080C80CFA7B D3DRMBoolean
 538684E0-BA3D-4BC0-ACA9-164AFF85C2A9 IVdsDiskPartitionMF
 538E61DE-7D46-5DA7-94AB-ACEDE6A12B9E IToggleSplitButtonAutomationPeer
 539698A0-CDCA-11CF-A5EB-00AA0047A063 IActiveScriptSiteInterruptPoll
@@ -5752,6 +5885,8 @@
 540BD122-5C83-4DEC-B396-EA62A2697FDF IWSDHttpMessageParameters
 541C8D6C-0283-4581-B945-2A64C28A0646 ISelectorItem
 542C5C4D-7596-434F-8C93-4EFA6766F267 ID3D12VideoProcessCommandList1
+5434E72D-6D66-11D1-BB0B-0000F875865A CDirect3DRMClippedVisual
+5434E733-6D66-11D1-BB0B-0000F875865A IDirect3DRMClippedVisual
 543A1BD9-6D0B-42B2-BD69-BC8FAE0F782C IAltimeterReading2
 543D66AC-FAF0-4560-84A5-0E4ACEC939E4 IPlayReadyRevocationServiceRequest
 543FA6D1-BD78-4866-ADDA-24314BC65DEA ICapturedFrame2
@@ -5759,6 +5894,7 @@
 5440837F-4BFF-4AE5-A1B1-7722ECC6332A AccStore
 544C8934-86D8-4991-8E75-ECE8A43B6B6D IProcessLauncherResult
 5453EF7E-7BDE-4B76-BB5D-48F79CFC8C0B ICreateAudioGraphResult
+5454E9A0-DB65-11CE-921C-00AA006C4972 IDirectPlay
 545B3A48-3283-4F62-866F-A62D8F598F9F IMFVideoSampleAllocatorEx
 54615221-A3C1-4CED-B62F-8C60523619AD IPhoneCallActivatedEventArgs
 546717C1-E8DF-4593-9639-97595FDF8310 IContentLinkInvokedEventArgs
@@ -5770,6 +5906,7 @@
 548D6B12-09FA-40E0-9069-5DCD589A52C9 ID3D12Debug5
 548DB883-C384-45C1-8AE8-A378C4EC486C ISyndicationItem
 5498D33A-E686-4935-A567-7CA77AD20FA4 INDLicenseFetchDescriptor
+549F498B-BFEB-11D1-8ED8-00A0C967A482 IDirect3DRMDevice3
 54A2CB2D-9A0C-48B6-8A50-9ABB69EE2D02 IUpdateDownloadContent
 54A8F188-9EBD-4795-AD16-9B4945119636 IWebBrowserEventsService
 54A9D260-8860-423F-B6FA-D77716F647A7 IHttpDiagnosticSourceLocation
@@ -5786,6 +5923,7 @@
 54DFD981-1968-4CA3-B958-E595D16505EB IEasClientDeviceInformation
 54E633FE-96E0-41E8-9832-1298897C2AAF IWebAuthenticationCoreManagerStatics4
 54EF13A1-BF26-47B5-97DD-DE779B7CAD58 ICredentialFactory
+54F39980-DA60-11CF-9B06-00A0C903A3B8 DDVPTYPE_E_HREFH_VREFH
 54FCB24B-E18E-47A2-B4D3-ECCBE77599A2 IInvokeProvider
 5500B84D-C79A-4EF8-B909-722E18E3935D IEmailRecipientFactory
 55050859-4024-474C-87F5-6472EAEE44EA ID3D12GraphicsCommandList5
@@ -5817,6 +5955,7 @@
 556189F2-2774-4DF6-9FA5-209F2BFB12F7 IUploadOperation2
 5563EBB1-3EF2-42C5-9C6D-1CF5DCC041FF IWriteableBitmapFactory
 556BA72E-CA9D-4211-B143-1CEDC86EB2FE IIsolatedWindowsEnvironmentShareFolderResult
+55728220-D33C-11CF-BFC7-444553540000 Key
 557EB5E5-54D0-5CCD-9FC5-ED33357F8A9F IScreenReaderPositionChangedEventArgs
 557F5244-92F8-4150-B730-E6346E8F50D1 ISectionsInViewChangedEventArgsFactory
 5581F42A-D3DF-4D93-A37D-C41A616C6D01 ISystemIdentificationStatics
@@ -5957,6 +6096,8 @@
 57D42F01-D762-4680-9893-326B08F22652 IWorkspaceBrokerAx2
 57D60D3B-EF1A-4E8E-839B-D36EF3A503E0 IContentLinkProviderFactory
 57D693AD-0C74-54DD-B110-1EE43FABADD9 IFlyoutShowOptions
+57D7C6BC-2356-11D3-8E9D-00C04F6844AE IDirectInputDevice7A
+57D7C6BD-2356-11D3-8E9D-00C04F6844AE IDirectInputDevice7W
 57DBE1A0-DE25-11D2-AFDD-00105A2799B5 ITfLangBarItemSink
 57EDB688-0C5E-416E-88B5-39B32D25C47D IAllJoynAboutDataViewStatics
 57EE6C15-1870-11D2-A689-00C04FA3490A ISMTPServerInternal
@@ -5994,6 +6135,7 @@
 58AA07A8-FED6-46F7-B4C5-FB1BA72108D6 IXmlLoadSettings
 58B498C2-7E4B-404C-9FC5-5DE8B40EBAE3 IMagnetometerDeviceId
 58BBB066-C2EA-5499-8150-40FAA75F6BB5 ISplitButton
+58C2B4D0-46E7-11D1-89AC-00A0C9054129 DirectMusicSynth
 58CC4352-1CA3-48E5-9864-1DA4D6E0D60F IGPMGPO
 58CD7703-8D87-4249-9B58-F6598CC9644E IKeyCredentialRetrievalResult
 58D3573B-F52C-45BE-988B-A5869564873C IUIElementStatics
@@ -6007,13 +6149,21 @@
 590EA907-D2EF-4C1A-9429-B4B15D64FD56 IMediaTransportControlsStatics
 591209C7-767B-42B2-9FBA-44EE4615F2C7 ApplicationAssociationRegistration
 59140509-8BC9-4EB4-B636-DABDC4F46F66 ISuspendingDeferral
+59163DE0-6D43-11CF-AC4A-0000C03825A1 IDirect3DRMUserVisual
 591720B4-7472-5218-8B39-DFFE615AE6DA IAnimationPropertyInfo2
 59213F4F-7346-49E5-B120-80555987A148 IUIAutomationRangeValuePattern
 59287F5E-BC81-4FCA-A7F1-E5A8ECDB1D69 ISyncMgrConflictFolder
 592E3B55-DE08-44E7-AC3B-B9B9A169583A IXboxLiveEndpointPairStateChangedEventArgs
 59308CEA-1E49-4921-BD88-A2878D07E30E IHyperlinkStatics5
 59359F4F-0862-53A3-A3B3-C932FB718CDC IPerformLocalActionRequestedEventArgs
+593817A0-7DB3-11CF-A2DE-00AA00B93356 DirectDrawClipper
 5938B49C-684A-48FA-8569-6BED64CA970D IDot11ExtUIProperty
+5944E662-AA8A-11CF-BFC7-444553540000 IDirectInput2A
+5944E663-AA8A-11CF-BFC7-444553540000 IDirectInput2W
+5944E680-C92E-11CF-BFC7-444553540000 IDirectInputDeviceA
+5944E681-C92E-11CF-BFC7-444553540000 IDirectInputDeviceW
+5944E682-C92E-11CF-BFC7-444553540000 IDirectInputDevice2A
+5944E683-C92E-11CF-BFC7-444553540000 IDirectInputDevice2W
 594572C2-82AA-410B-9E55-FD8E2C98862D IPivotHeaderItem
 594EBE78-55EA-4A88-A50D-3402AE1F9CF2 IThreadPoolTimer
 594F31D0-7F19-11D0-B194-00A0C90DC8BF IRpcChannelBuffer2
@@ -6035,6 +6185,7 @@
 59AB8AE7-F184-4AB1-B0B0-35C8BF0654B2 IMapControlBusinessLandmarkRightTappedEventArgs
 59AC2A57-B32A-479E-8A61-4B7F2E9E7EA0 IContainerEncodingProperties
 59B893A9-73BC-52B5-BA41-82511B2CB46C ISystemButtonEventController
+59B95640-9667-11D0-A77D-0000F803ABFC DPAID_LobbyProvider
 59BA2FBB-24CB-4C02-A4C4-8294569D54F1 ILauncherStatics2
 59BB79ED-5435-456C-A8D1-6680D56AAD1E IOemSlotStateChange
 59BC1661-C182-49AF-9526-44B88E628455 IHyperlinkButtonAutomationPeerFactory
@@ -6551,6 +6702,7 @@
 63686821-58B9-47EE-93D9-F0F741CF43F0 IProtectedContentRevokedEventArgs
 6369BF15-0A96-577A-8FF6-C35904DFA8F3 IRemoteSystemApp2
 6369C758-5644-41E2-97AF-8EF56A25E225 ILowLagMediaRecording2
+636B9F10-0C7D-11D1-95B2-0020AFDC7421 DirectMusic
 63711A9D-4118-45A6-ACB6-22C49D019F40 IStorageProviderGetPathForContentUriResult
 6374EA4C-1B3B-4001-94D9-DD225330FA40 IAudioStateMonitorStatics
 63856340-5E87-4C10-86D1-6DF097A6C6A8 IMediaFrameFormat2
@@ -6602,6 +6754,8 @@
 647E03B7-036A-4DEA-9540-1DD7FD1266F1 IUIElementStatics6
 648A2B4D-53CA-4B5A-AA8E-3CC7440F4A67 IFrame3
 64961751-0835-43C0-8FFE-D57686530E64 IExplorerCommandProvider
+6496ABA1-61B0-11D2-AFA6-00AA0024D8B6 DMUS_PROP_XG_Capable
+6496ABA2-61B0-11D2-AFA6-00AA0024D8B6 DMUS_PROP_GS_Capable
 64976BFA-FB61-4041-9069-8C9A5F659BEB IMFByteStreamTimeSeek
 64A1CBF0-3A1A-4461-9158-376969693950 IFileIsInUse
 64AB4BB7-111E-11D1-8F79-00C04FC2FBE1 ShellUIHelper
@@ -6639,6 +6793,7 @@
 652B6900-82DE-11DF-8901-09B0DFD72085 IRdServer
 652C208C-8CCF-4399-BD3E-5A015AA1BC03 IContentControl2
 65329C05-FB5A-4567-9D55-5CDFBADA2739 IMarkupExtensionFactory
+6536115A-7B2D-11D2-BA18-0000F875AC12 IDirectMusic
 6539864B-52FA-4E35-907C-834CAC417B50 ITensorInt32BitStatics
 6544A28A-F68D-47AC-91EF-16B2B36AA3EE IFhReassociation
 6548CA01-86DE-5BE1-6582-807FCF4C95CF IPerceptionDepthCorrelatedCameraIntrinsics
@@ -6676,6 +6831,7 @@
 65ACEADC-2F0B-4F43-9F4D-811865D8CEAD WdsTransportServicePolicy
 65AED1AE-B95B-450C-882B-6255187F397E IPackage4
 65B46685-60DA-5E51-91A2-B38C4763B872 IVideoEncodingPropertiesStatics3
+65B76FA5-FF37-11D2-814E-00C04FA36E58 SeedVariations
 65B912EB-80B9-44F9-9C1E-E120F6D92838 IMediaSourceAppServiceConnectionFactory
 65C19E5C-AA7E-4B91-8944-91E0E5572797 IWdsTransportServicePolicy2
 65C66FAC-FE8C-46D4-9119-57B8410311D5 IChatMessageStoreChangedEventArgs
@@ -6801,6 +6957,7 @@
 68510F1F-88E3-477A-A2FA-569F5F1F9BD5 IInkStrokeRenderingSegment
 68545726-2DC7-51BF-B470-0B344CC2A1BC ITensorFloat16BitStatics2
 685AC88B-79B2-4B76-9E03-A876223DFE72 IPaymentItem
+685BC400-9D2C-11CF-A9CD-00AA006886E3 DPSPIPX
 685D4176-F6E0-4569-B1AA-215C0FF5B257 IIsolatedWindowsEnvironmentLaunchFileResult
 6863AB72-4997-47F9-87AD-37EFB75993EA IRepeatBehaviorHelper
 686CD241-4330-4C77-ACE4-0334968B1768 IBitmapEncoderWithSoftwareBitmap
@@ -6818,6 +6975,7 @@
 68961E68-832B-41EA-BC91-63593F3E70E3 IMediaSample2Config
 6898D761-56C9-4F17-BE31-66FDA94BA159 IEmailMailboxUpdateMeetingResponseRequestEventArgs
 689E0708-7EEF-483F-963F-DA938818E073 ISoftwareBitmap
+68A04844-D13D-11D1-AFA6-00AA0024D8B6 IDirectMusicGetLoader
 68A0C487-2B4F-46C2-94A1-6CE685183634 ITsSbPlacementNotifySink
 68A7A829-2D5F-4653-8BB0-BD2FA6BD2D87 IWebAccountManagerStatics2
 68B4774D-63E5-4AF0-A58B-3E535B94E8C9 ILampArrayCustomEffectFactory
@@ -7134,6 +7292,13 @@
 6F10711F-729B-41E5-93B8-F21D0F818DF1 IStaticPortMapping
 6F13083E-BF62-4FE2-A95A-F08999669818 II2cProvider
 6F1578F2-6D77-4F8A-8C2F-6130B6395053 IIsoSpeedControl2
+6F1D2B60-D5A0-11CF-BFC7-444553540000 SysMouse
+6F1D2B61-D5A0-11CF-BFC7-444553540000 SysKeyboard
+6F1D2B70-D5A0-11CF-BFC7-444553540000 Joystick
+6F1D2B80-D5A0-11CF-BFC7-444553540000 SysMouseEm
+6F1D2B81-D5A0-11CF-BFC7-444553540000 SysMouseEm2
+6F1D2B82-D5A0-11CF-BFC7-444553540000 SysKeyboardEm
+6F1D2B83-D5A0-11CF-BFC7-444553540000 SysKeyboardEm2
 6F354338-3302-4D13-AD8D-ADCC0FF9E47C IContactRemovedEventArgs
 6F3837FC-8E3D-4522-9B0F-003DB8609851 IEasingColorKeyFrameStatics
 6F3DC998-DD47-433C-9234-560172D67A45 IAppInstallItem3
@@ -7164,6 +7329,8 @@
 6FBC7005-0455-4874-B8FF-7439450241A3 IUrlAccessor3
 6FBC7D67-F9B1-55E0-9D88-D3A5197A3F59 IDisplayTaskResult
 6FBD02FB-209F-46C4-B059-2ED15586A6AC ID3D11RasterizerState2
+6FC2CAE0-BC78-11D2-AFA6-00AA0024D8B6 IDirectMusicPerformance2
+6FC2CAE1-BC78-11D2-AFA6-00AA0024D8B6 IDirectMusic2
 6FCD1DC4-175F-5BD2-8DE5-2F2006A25ADF ILearningModelSessionOptions2
 6FCEE2D3-6D90-4F91-80E2-A5C7CAACA9D8 IVdsSubSystem
 6FD519E5-9E7E-4AB4-8BE3-C76B1BD4344A ISearchSuggestionsRequestedEventArgs
@@ -7419,6 +7586,7 @@
 7445F5FD-AA9D-4619-8D5D-C138A4EDE3E5 IWebAccountMonitor
 7449121C-382B-4705-8DA7-A795BA482013 IDisplayInformationStaticsInterop
 744930EB-8157-4A41-8494-6BF54C9B1D2B IMobileBroadbandUiccAppsResult
+745A17A0-74D3-11D0-B6FE-00A0C90F57DA HIDClass
 747B2F4F-7B25-56AA-8300-78D83B7AB2DB ITreeViewCollapsedEventArgs2
 7488C029-6A53-4258-A3E9-62DFF6784B6C IContactPickerStatics
 7489316F-77A0-433D-8ED5-4040280E8665 IAudioEncodingPropertiesStatics2
@@ -7502,6 +7670,7 @@
 761AA641-7BDA-11D1-8AA9-00C04FB951F3 IMimeMessageCallback
 761C8CFD-A19D-5A51-80F5-71676115483E IGeocoordinateSatelliteData2
 761CE752-73C1-4F44-BE75-43C49EC0D4D5 IAutomationPeer4
+762CCDA1-D916-11D0-BA39-00C04FD7ED67 DPLPROPERTY_MessagesSupported
 763070ED-D04E-4FAC-B233-600B42EDD469 IAudioGraphConnection
 7632E1F5-EE65-4DCA-87FD-84CD75F8838D IDXGIFactory5
 764651D0-38DE-11D4-A2A3-00104BD35090 IActiveScriptSIPInfo
@@ -7627,6 +7796,7 @@
 78DBF87B-F766-422B-A61C-C8C446BDB9AD ID3D12DeviceConfiguration
 78DCB116-818A-4CB5-BCA7-382CE6DDC90D IAutoSuggestBoxQuerySubmittedEventArgs
 78E591AF-B142-4EF0-8830-55BC2BE4F5EA IGameModeConfiguration
+78EC89A0-E0AF-11CF-9C4E-00A0C905425E DPAID_Phone
 78EDBE35-27A0-42E0-9CD3-738D2089DE9C ILowLightFusionResult
 78F30B81-48AA-11D2-9900-0000F803FF7A IDXTRoll
 78F8EF57-66C3-4E09-BD7C-E79B2004894D IUIAutomationGridItemPattern
@@ -7667,6 +7837,7 @@
 79B56888-7FEA-4690-B45D-38FD3C7849BE IBDA_Topology
 79B6B0E5-069A-4486-A99D-8DB772B9642F IRadialControllerConfigurationStatics
 79B9D5F2-879E-4B89-B798-79E47598030C ICoreWindow
+79BA9E00-B6EE-11D1-86BE-00C04FBF8FEF DirectMusicBand
 79BB7E8D-DCA9-4B5F-A448-37B13238ED76 IAppBarStatics
 79BBCF4E-CD66-4F1B-A8B6-CD6D2977C18D IAcrylicBrush
 79C2219E-44D2-4610-9735-9BEC83809ECF IBitmapCache
@@ -7977,6 +8148,8 @@
 7F027B22-1515-4E85-AA0D-026486580576 ID3D12VideoEncodeCommandList3
 7F0B3150-7B81-4AD4-98E3-7E9097094301 IBDA_NameValueService
 7F0C58E9-4F30-47BB-9693-D778E999C433 IHomePlugin
+7F0F21E0-BFE1-11D1-82C0-00A0C9697271 D3DRMStringProperty
+7F0F21E1-BFE1-11D1-82C0-00A0C9697271 D3DRMPropertyBag
 7F138A08-FBB7-4B09-8356-AA5F4E335C31 IPlayToSource
 7F29F23C-AA04-4B08-97B0-06D874CDABF5 IBackgroundTaskInstance4
 7F343202-E010-52C4-AF0C-04A8F1E033DA IHumanPresenceSettingsStatics
@@ -7984,6 +8157,7 @@
 7F5AC5A5-49DB-379F-9429-32A2FAF34FA6 IHeadPose
 7F5BA925-CDB1-4D11-A71F-339EB7E709FD IVssHardwareSnapshotProviderEx
 7F5D1F2B-E04A-4D10-B31A-A55C922CC2FB IVoipCallCoordinatorStatics
+7F5D5EA0-D53A-11D1-82C0-00A0C9697271 D3DRMRightHanded
 7F5EE992-7694-4E6C-A51B-E34BF43DE743 ISupportIncrementalLoading
 7F5FD609-BC88-41D4-88FA-3DE6704D428E IResourceCollection
 7F605B87-4832-4B5F-AE3D-412FAAB37D34 IMediaFrameSourceGroup
@@ -8106,6 +8280,7 @@
 81EA942C-4F09-4406-A538-838D9B14B7E6 IAccountsSettingsPane
 81F04AD1-4194-4D7D-86D6-11813CEC163C IFeedFolder
 81F21AC0-F661-4708-BA4F-D386BD0D622E IContactManagerStatics
+81F75BC5-4E5D-11D2-BCC7-00A0C922E6EB NOTIFICATION_PERFORMANCE
 81F8AD60-1CE8-469D-A667-16E37CEF8BA9 ISetterBaseFactory
 81FD76E2-0B26-48D9-98CE-27461C669D6C IStorePreviewSkuInfo
 82001A48-09C0-434F-8189-141C66DF362F IInteractionTrackerVector2InertiaNaturalMotionStatics
@@ -8327,6 +8502,7 @@
 8648B31E-6E52-5259-BBAD-242A69DC38C8 IGyrometerDataThreshold
 864D0B26-3A37-4251-9A02-B231198DE060 ISystemDataPathsInterop
 865BDA7A-21D8-4573-8F32-928A1B0641F6 IBackgroundTaskInstance
+865FD372-9F67-11D2-872A-00600893B1BD DMUS_PROP_INSTRUMENT2
 8660B4D6-969B-4E3C-AC3A-8B042EE4EE63 IAppBroadcastServices
 8664DA16-DDA2-42AC-926A-C18F9127C302 IDVBTLocator
 86664478-8029-46FE-A789-1C2F3E2FFB5C IStorageItemProperties
@@ -8335,6 +8511,7 @@
 866EF088-DE68-4ABF-88A1-F90D16274192 IRemoteSystemSessionParticipantRemovedEventArgs
 866F65D5-EA97-42AB-9288-9C01AEBC7A97 ITypography
 868CE85C-8EA9-4F55-AB82-B009A910A805 IMFPresentationClock
+86B44E25-9C82-11D1-BB0B-00A0C981A0A6 IDirect3DRMShadow2
 86B4D4D1-B2FE-4E34-A81D-66640300454F IWordsSegmenter
 86BEC222-30F2-47E0-9F25-60D11CD75C28 ApplicationDocumentLists
 86C14003-4D6B-4EF3-A7B4-0506663B2E68 ApplicationDestinations
@@ -8484,6 +8661,8 @@
 8948BA79-A243-4AA3-A294-1F98BD0FF580 IInkAnalysisResult
 894AD3B0-EF97-11CE-9BC9-00AA00608E01 IOleUndoUnit
 894D8C55-BDDF-11D1-B85D-00C04FB93981 ISyncMgrRegister
+89521360-AA8A-11CF-BFC7-444553540000 IDirectInputA
+89521361-AA8A-11CF-BFC7-444553540000 IDirectInputW
 895491E2-E701-46A9-9A1F-8D3480ED867A ID3D12VideoEncodeCommandList2
 89572027-4C48-4700-8076-497BA73D9C18 ISlider
 895894DF-5A9F-4A1F-85EC-6F3C1B6DCB89 IMenuFlyoutPresenter2
@@ -8521,6 +8700,7 @@
 8A523716-0548-11D2-A894-0000F8084F96 IPropFindResponse
 8A58CDC0-CBDC-11D0-A9F8-00AA00685C74 CEventMetabaseDatabaseManager
 8A610DC6-D72D-4CA8-9F66-556F35B513DA IAccessKeyDisplayDismissedEventArgs
+8A63C360-997D-11D0-941C-0080C80CFA7B D3DRMProgressiveMesh
 8A66A210-B78B-4D99-88E2-C306A817C925 IGPMGPO2
 8A674B48-1F63-11D3-B64C-00C04F79498E ICreatePropBagOnRegKey
 8A674B49-1F63-11D3-B64C-00C04F79498E CreatePropBagOnRegKey
@@ -8912,6 +9092,7 @@
 926130B8-9955-4C74-BDC1-7CD0DECF9B03 IAnalyticsVersionInfo
 9264545E-E158-4E74-8899-689160BD2F8C IMatrix3DHelperStatics
 926749FA-2615-4987-8845-C33E65F2B957 UIRibbonFramework
+92783220-DA60-11CF-9B06-00A0C903A3B8 DDVPTYPE_E_HREFH_VREFL
 9285B776-2E7B-4BC0-B53E-580EB6FA967F ISpeechObjectTokens
 928BF3D0-CF7C-4AB0-A7DC-6DC5BCD44252 IAdvertisingManagerForUser
 928C088B-76B9-4C6B-A536-A590853876CD IDirect3DSwapChain8
@@ -9194,6 +9375,7 @@
 97EC2EA4-0E42-4937-97AC-9D6D328824E1 IMFShutdown
 97F0F1A6-E04E-4EA4-B4F9-B0D0E8D90F5D IDiaStackWalkFrame
 980C93EA-241F-4F6A-A539-9D185C8CF18A IRichEditBox7
+98116AA0-BDBA-11D1-82C0-00A0C9697271 D3DRMExternalVisual
 98141C1D-0D0E-4175-BBE2-6BFF455842A7 IUIAutomationElement5
 981611AD-A144-4C83-9890-F30E26D658AB ID3D12VideoDevice1
 9829B0A0-5A30-5EED-A3A3-1472C53B0A66 IToggleSplitButtonAutomationPeerFactory
@@ -9309,6 +9491,8 @@
 9A3FCCB2-B9BB-404B-A22B-66DCCD1250C6 IWebViewControlContentLoadingEventArgs
 9A438A6A-7F4C-452E-88BD-0148C6387A2C IPlayReadyContentHeader
 9A43A843-0831-11D1-817F-0000F87557DB IDXTComposite
+9A4CB684-236D-11D3-8E9D-00C04F6844AE IDirectInput7A
+9A4CB685-236D-11D3-8E9D-00C04F6844AE IDirectInput7W
 9A512C86-AC6E-4A8E-B1A4-FCEF363F6E64 IRDPSRAPIPerfCounterLoggingManager
 9A53B261-C70C-42BE-B2F3-F42AA97D34E5 ITileFlyoutNotification
 9A586B89-D5D0-4FBE-BD9D-B5B3636811B4 IHttpCacheDirectiveHeaderValueCollection
@@ -9367,6 +9551,7 @@
 9B6E051C-5DDD-4321-9070-FE2ACB55E794 IPropertyEnumType2
 9B6E1AF0-1A92-40F3-A59D-F36AC1F728B7 IGPMStatusMsgCollection
 9B713FD7-839D-4A74-8732-45466FC044B5 ISpatialNavigationUpdatedEventArgs
+9B74ED7A-BBEF-11D2-9F8E-0000F8080835 ID3DXContext
 9B78F0E6-3E05-4A5B-B2E8-E743A8956B65 RDPSession
 9B7E472A-0DE4-4640-BFF3-84D38A051C31 ISAXLocator
 9B822734-A3C1-542A-B2F4-0E32B773FB07 IAttachableInputObject
@@ -9456,6 +9641,7 @@
 9D2F920D-A588-43C6-89D6-5AD322AF006A IMediaCaptureDeviceExclusiveControlStatusChangedEventArgs
 9D416F9C-E184-45B2-A4F0-CE517F719E9B IDiaPropertyStorage
 9D457B9B-019F-4266-8872-215F198F6A9D IGridLengthHelperStatics
+9D460580-A822-11CF-960C-0080C7534E82 IDirectPlay2A
 9D5E227A-4430-4161-88B3-3ECA6BB16E19 ID3D12Resource1
 9D66FE6C-A4D7-4C23-9E85-675F304C252D IResourceManager2
 9D697B05-D4F2-4C7A-9DFE-8D6CD0B3EE84 IMediaStreamSourceSampleRenderedEventArgs
@@ -9603,6 +9789,7 @@ A06D048D-2B8D-4754-AD5A-D0871193A956 IInkAnalysisLine
 A06EB39A-50DA-425B-8C31-4EECD6C270F3 ID3DDestructionNotifier
 A06F9F05-3BE9-4763-98A8-094FC672E488 IXpsOMSolidColorBrush
 A072F1D6-0286-4E1F-85F2-17A2948456EC IRTCSessionPortManagement
+A07A02E0-DA60-11CF-9B06-00A0C903A3B8 DDVPTYPE_E_HREFL_VREFH
 A08CE4D0-FA25-44AB-B57C-C7B1C323E0B9 IExplorerCommand
 A09513ED-C709-4D21-BD7B-5F34C47F3947 IControlChangeNotify
 A09691DF-9824-41FE-B530-B0D8990E64C1 IControlOverrides
@@ -9743,6 +9930,16 @@ A359DEC5-E813-4834-8A2A-BA7F1D777D76 IWbemBackupRestoreEx
 A35F2DEA-098F-4EBD-984C-2BD4C3C8CE0A IPBDA_EIT
 A36824E3-3D81-4CE5-AA51-8B410F602FCD IStyleFactory
 A36C88F2-AFDC-4775-855D-D4AC0A35FC55 IVpnChannelActivityEventArgs
+A36D02E0-C9F3-11CF-BFC7-444553540000 XAxis
+A36D02E1-C9F3-11CF-BFC7-444553540000 YAxis
+A36D02E2-C9F3-11CF-BFC7-444553540000 ZAxis
+A36D02E3-C9F3-11CF-BFC7-444553540000 RzAxis
+A36D02E4-C9F3-11CF-BFC7-444553540000 Slider
+A36D02F0-C9F3-11CF-BFC7-444553540000 Button
+A36D02F2-C9F3-11CF-BFC7-444553540000 POV
+A36D02F3-C9F3-11CF-BFC7-444553540000 Unknown
+A36D02F4-C9F3-11CF-BFC7-444553540000 RxAxis
+A36D02F5-C9F3-11CF-BFC7-444553540000 RyAxis
 A36D608C-C29D-4B94-B086-7E9FF7BD84A0 IAppointmentCalendarUpdateMeetingResponseRequest
 A37251B7-6B44-473D-8F24-F703D6C0EC44 ILowLagPhotoCapture
 A372ACD1-3BEF-4BBD-8FFB-CB3E2B416AF8 DISPID_SpeechRecognizer
@@ -9763,11 +9960,14 @@ A39EE748-6A27-4817-A6F2-13914BEF5890 IUri
 A3A2F4CF-258B-420C-B8AB-8EDD601ECF50 ILightSensorReadingChangedEventArgs
 A3A416A4-0F64-11D3-B5B7-00C04FC324A1 ITfFnPlayBack
 A3A4A501-6A2D-52D7-B04B-C435BAEE0115 ITensorBooleanStatics2
+A3A80D01-6E12-11CF-AC4A-0000C03825A1 IDirect3DRMMesh
+A3A80D02-6E12-11CF-AC4A-0000C03825A1 IDirect3DRMMeshBuilder
 A3A9FEFF-8021-441B-9A46-E7F0137B7981 IAudioMediaFrame
 A3ACFB0E-83FF-11D2-9E14-00C04FA322BA IMailTransportOnPreCategorize
 A3AD50FB-9BDB-49E3-A843-6C76520FBF5D ITfCandidateList
 A3AD8D93-79F8-4958-A3C8-980DEFB83D15 IMediaTransportControlsAutomationPeer
 A3AFB9CC-B653-4741-86AB-F0470EC1384C _Application
+A3AFDCC7-D3EE-11D1-BC8D-00A0C922E6EB IDirectMusicSegmentState
 A3B14589-9174-49A8-89A3-06A1AE2B9BA7 ICategorizer
 A3B152DF-7A90-4218-AC54-9830BEE8C0B6 ITuneRequestInfo
 A3B493BB-CB4F-4E6B-A33D-F11E7A295EFA IInkToolbarStatics
@@ -9788,6 +9988,9 @@ A417BC0F-7BE1-11CE-AD82-00AA006EC559 IReplStore
 A420BC4A-6E2E-4AF5-B556-355AE0CD4F29 IStreamSocketStatics
 A4216A22-1503-4EBF-8FF5-2DE84584A8AA IInkStrokesErasedEventArgs
 A424801B-C5EA-4D45-A0EC-4B791F041A89 ICompass3
+A42790E0-7810-11CF-8F52-0040333594A3 D3DRMGuid
+A42790E1-7810-11CF-8F52-0040333594A3 D3DRMTextureFilename
+A42790E2-7810-11CF-8F52-0040333594A3 D3DRMTextureReference
 A428100A-E646-4BD6-AA14-6087BDC08CD5 IIsdbDataContentDescriptor
 A4296F0B-DFF8-467C-9315-6F6DB93EE2BA IMapElementStatics4
 A42B3316-8415-40DC-8C44-98361D235430 ISystemMediaProperties
@@ -10023,6 +10226,7 @@ A8912263-2951-4F58-A9C5-5A134EAA7F07 IControl
 A89C9712-48E0-4240-95B9-0DFD0826A8D3 IFlyoutBase3
 A8A47FDE-CA91-44D2-B897-3AA28A43B2B7 ITsSbResourceNotificationEx
 A8A6DEF0-7AA8-5326-B7C4-88AE25013362 IInkToolbar3
+A8A98BA0-C5E5-11CF-B941-0080C80CFA7B D3DRMLightAttenuation
 A8ABC4E9-A84A-47A9-80B3-C5D9B172A961 IPortableDeviceServiceManager
 A8AF82F3-D803-5C25-B48D-AE0E73B16D8E ITreeViewItemStatics2
 A8B11756-A39F-4E97-8C7D-C1EA8DD633C5 ITransformProvider2
@@ -10377,6 +10581,9 @@ AEC47CA3-4477-4B04-A06F-2C1C5182FE9D IMediaCapturePauseResult
 AEC57C6F-47B4-546B-987C-E73FA791FE3A IDisplayTarget
 AEC63FDA-7A97-4944-B35C-6C6DF8085CC3 IMFMediaKeySystemAccess
 AEC7CAA7-9AC6-4BF5-8B91-5C1ADF0D7D00 IIsolatedWindowsEnvironmentShareFileResult
+AED22740-B31F-11CF-9DD5-00AA00A71A2F D3DRMLightUmbra
+AED22741-B31F-11CF-9DD5-00AA00A71A2F D3DRMLightPenumbra
+AED22742-B31F-11CF-9DD5-00AA00A71A2F D3DRMLightRange
 AED6483C-3304-11D2-86F1-006008B0E5D2 ITPluggableTerminalInitialization
 AEDA51C0-9025-4983-9012-DE597B88B089 IMFCaptureEngineOnEventCallback
 AEDAE97E-D7EE-4796-B960-7F092AE844AB IActiveScriptSiteUIControl
@@ -10403,9 +10610,11 @@ AF230D27-BABA-4E42-ACED-F524F22CFCE2 INetFwRule
 AF2440F6-8AFC-47D0-9A7F-396A0ACFB43D INamedEntityCollector
 AF279B2F-86EB-11D1-81BF-0000F87557DB IDXTWipe
 AF338AE5-CF64-423C-85E5-F3E72448FB23 IApplicationViewStatics2
+AF359780-6BA3-11CF-AC4A-0000C03825A1 IDirect3DRMShadow
 AF40329B-B544-469B-86B9-AC3D466FEA36 IHttpRequestHeaderCollection
 AF404424-26DF-44F4-8714-9359249B62D3 IDoubleTappedRoutedEventArgs
 AF45AF49-D6AA-407D-BF87-3912236E9D94 IPrintSchemaDisplayableElement
+AF465C71-9588-11CF-A020-00AA006157AC IDirectPlayLobby
 AF4BB9EE-8984-49B7-81DF-3F35994B95EB IBezierSegment
 AF4F3F91-564C-4DDC-B8B5-973447627C65 IUserConsentVerifierStatics
 AF5207D9-C433-4764-B3FD-8FA6BFBCBADC IResourceCandidate
@@ -10442,6 +10651,7 @@ B006BAC0-751B-4D55-92CB-613EC1BDF5D0 IExpandCollapsePatternIdentifiers
 B014F270-9808-4ADC-B452-60CDA54100C1 ITextBlock4
 B01726FF-969A-40D0-9F04-76084CE6D7B7 ISwipeItems
 B01B6BE1-6F4E-49E2-AC01-559D62BEB3A9 IAudioDeviceInputNode
+B0210780-89CD-11D0-AF08-00A0C925CD16 DirectSoundCapture
 B0210781-89CD-11D0-AF08-00A0C925CD16 IDirectSoundCapture
 B0210782-89CD-11D0-AF08-00A0C925CD16 IDirectSoundCaptureBuffer
 B0210783-89CD-11D0-AF08-00A0C925CD16 IDirectSoundNotify
@@ -10449,6 +10659,8 @@ B022D3AE-884D-4D85-B146-53320E76EF62 IWdsTransportTftpClient
 B0234C3E-CD6F-4CB6-A611-61FD505F3ED1 IJumpList
 B03413D2-0723-4C4E-9E08-2E9C1FF3772B TS_SELECTION_ANCHOR
 B038876A-9C4D-4402-AC56-A223487DC6C8 IContentPresenter3
+B03E1181-8095-11D2-A1EF-00600833DBD8 DLSID_ManufacturersID
+B03E1182-8095-11D2-A1EF-00600833DBD8 DLSID_ProductID
 B04F29A1-BD16-48F6-A511-9C2763641331 IDragCompletedEventArgs
 B054A561-9833-4AED-9717-4348B21A24B3 IMobileBroadbandRadioStateChange
 B068E05F-C41F-4AC9-AFAB-827AB6248F9A IColorOverrideSettingsStatics
@@ -10658,6 +10870,7 @@ B413A36A-E294-4591-A5A6-64891A828153 IGattServiceProviderTriggerStatics
 B41DA63B-A4B8-4036-89C5-83C31266BA49 IWorkspace
 B4202F92-1611-4E05-B6EF-DFD737AE33B0 IGameBarServicesTargetInfo
 B4211CFA-51B9-4A9F-AB78-DB99B2BB678E IDirect3DBaseTexture8
+B4319322-D20D-11D0-BA39-00C04FD7ED67 DPLPROPERTY_PlayerGuid
 B43BBCF9-65A6-42DD-BAC0-8C6741E7F5A4 IAppxManifestPackageDependenciesEnumerator
 B440BF07-068F-4537-84C6-4ECBE019E1F4 ICompositionViewBox
 B440C7C3-02B4-4980-9342-25DAE1C0F188 IDragEventArgs
@@ -10970,6 +11183,7 @@ BA2743A1-07E0-48EF-84B6-9A2ED023CA6C IMFMediaEngineWebSupport
 BA3A37E8-33DA-4749-8DA0-07FA34DA7944 IRDPSRAPIAttendeeManager
 BA468C55-9956-4FB1-A59D-52A7DD7CC6AA ITfLangBarItemMgr
 BA4B6526-1A35-4635-8B56-3EC612746A8C IESOpenMmiEvent
+BA5A7A70-9DBF-11D0-9CC1-00A0C905425E DPAID_PhoneW
 BA6295C3-312E-544F-9FD5-1F81B21F3649 IStorageProviderQuotaUI
 BA6CB54B-7D75-4535-B30D-A81A00B637A4 ITextHighlighter
 BA6EBE88-7570-4E69-ACCF-5611AD015F88 IMediaEncodingProfile3
@@ -11412,6 +11626,9 @@ C22615D2-D318-4DA2-8422-1FCAF77B10E4 IWebApplicationNavigationEvents
 C227C2CA-26F4-5960-98D5-919149D1B525 ISplitButtonClickEventArgs
 C22DF897-A3E8-5051-B3CC-B5F0F5262950 IAutoSuggestBox4
 C23FC28D-C55F-4720-8B32-91F73C2BD5D1 SpPhraseInfoBuilder
+C241333F-1C1B-11D2-94F5-00C04FC28ACA DS3DALG_NO_VIRTUALIZATION
+C2413340-1C1B-11D2-94F5-00C04FC28ACA DS3DALG_HRTF_FULL
+C2413342-1C1B-11D2-94F5-00C04FC28ACA DS3DALG_HRTF_LIGHT
 C246FF23-D80D-44DE-8DB9-0D815E269AC0 IHoldingRoutedEventArgs
 C247E8F7-ADCC-440F-B123-33788A40525A IValuePatternIdentifiersStatics
 C2507D83-6E0A-4A8A-BDF8-1948874854BA ICoreTextServicesManager
@@ -11491,6 +11708,7 @@ C3CA36B9-0793-4ED0-BBF4-9FF4E0F98F80 IObjectModelProvider
 C3D3D11E-FA26-4CE7-A4ED-FF948F01B7C0 IListViewItemPresenterStatics3
 C3D9CBE0-3FF6-4FB3-B262-50D435C55423 IAudioGraphUnrecoverableErrorOccurredEventArgs
 C3DA6FB7-B744-4B45-B0B8-223A0938D0DC IApplicationData
+C3DFBD60-3988-11D0-9EC2-0000C0291AC3 IDirect3DRMFrame2
 C3E12EB5-7D8D-44F8-B6DD-0E77B34D6DE4 IInitializeWithPropertyStore
 C3E4F353-433F-43D6-89A1-6A62A7054C3D ISpeechPhoneConverter
 C3E847FF-0EBB-4A53-8C54-DEC24926CB2C ILoggingChannelOptions
@@ -11530,6 +11748,7 @@ C48D8917-95A8-47B8-A517-BF891A6C039B IItemsControlAutomationPeer2
 C497789B-B499-4D69-B5C2-AE9D5D6D594E IWebView3
 C49E32C6-BC8B-11D2-85D4-00105A1F8304 WbemBackupRestore
 C49E32C7-BC8B-11D2-85D4-00105A1F8304 IWbemBackupRestore
+C4A54DA0-E0AF-11CF-9C4E-00A0C905425E DPAID_INet
 C4A9F225-9DB7-4A7D-B6D1-F74EDB9293C2 IStyle
 C4AB3503-AE7F-4387-AD19-00F150E48EBB ICmsDetachedSignatureFactory
 C4AEF0C6-16A3-484B-87F5-6528B8F04A47 ITimelineMarkerStatics
@@ -11560,6 +11779,7 @@ C4F0E950-3BFF-53D6-A0F8-514EA3705C66 IHumanPresenceSensorReading2
 C4F29CAB-60AD-4F24-BD27-9D69C3127C9A IProjectionFactory
 C4F3F229-1938-495C-B4D9-6DE4B08D4811 IInkStrokesCollectedEventArgs
 C4FEA4FD-7AF3-4952-9FD9-9E0987FC4F29 IFlipView2
+C5016CC0-D273-11CE-AC48-0000C03825A1 IDirect3DRMWinDevice
 C5143AE8-1B03-46F8-B694-A523E83CFCB6 IContactChangedDeferral
 C5194341-9742-4ED5-823D-FC1401148764 IKnownFoldersStatics3
 C52D32DD-F2B4-4052-8502-EC4305ECB71F IPrinterBidiSetRequestCallback
@@ -12051,6 +12271,7 @@ CD8F23C1-8F61-4916-909D-55BDD0918753 IFileOperation2
 CD91D690-A7E8-4265-9B38-8BB3BBABA7DE IUIManagerEventSink
 CD937743-7A55-4D3B-9021-F22E022D09C5 ISmsWapMessage
 CD951653-C2F1-4498-8B78-5FB4CD3640DD ICastingConnection
+CDA8D611-684A-11D2-871E-00600893B1BD DMUS_PROP_Effects
 CDAECE56-4EDF-43DF-B113-88E4556FA1BB IProofOfPossessionCookieInfoManager
 CDB780D1-E7BB-4D1A-9339-34DC41149F9B IProtectedContainerImportResult
 CDBEC9C0-7A68-11D1-88F9-0080C7D771BF CEventClass
@@ -12137,6 +12358,7 @@ CEB78D07-D0A8-4703-A091-C8C2C0915BC3 IWebSocketClosedEventArgs
 CEB78D07-D0A8-4703-A091-C8C2C0915BC4 IVpnPlugIn
 CEBFE864-DBEA-4404-8E6E-DE55ADA75239 IRepositionThemeTransition2
 CECBD2C3-A3A5-4749-9681-20E9161C6794 IWebApplicationHost
+CED153E7-3606-11D2-B9F9-0000F875AC12 IDirectMusicThru
 CED175E5-1935-4820-81BD-FF6AD00C9108 IVMRMixerBitmap9
 CEEE3B62-8F56-4056-869B-EF16917E3EFC MsftIsoImageManager
 CEEF98F1-E407-4128-8E73-D93D5F822786 IPaymentMediator2
@@ -12184,11 +12406,13 @@ CFA1658B-0123-4551-8891-89BDCC40322B ICompositorWithVisualSurface
 CFA59A45-28B1-4868-969E-FA8097FDC12A IOpcSignatureReferenceEnumerator
 CFA688F0-639E-4A47-83D7-6B7F5EBF7FED IHolographicQuadLayerInterop
 CFA7A973-A3B4-4494-B3FF-CBA94DB69699 IMapAddress
+CFA7CDC2-00A1-11D2-AAD5-0000F875AC12 DMUS_PROP_LegacyCaps
 CFB18DBB-9931-4119-8BDA-54A86FDFAC84 IDatePickerFlyout2
 CFB2A855-1C59-4B13-B1E5-16E29805F7C4 ICoreWindowDialogFactory
 CFC2F366-C3E8-40FD-8CC3-67E289004938 IProximitySensorReadingChangedEventArgs
 CFC399AF-D876-11D0-9C10-00C04FC99C8E XMLDocument
 CFCC809F-295D-42E8-9FFC-424B33C487E6 IHWEventHandler2
+CFCD4602-EB7B-11D2-A440-00A0C90629A8 ID3DXSimpleShape
 CFD57E3B-0256-585D-8B3F-FEAFB69D3320 ITextBlockStatics7
 CFDA55B0-5E3E-4289-932D-EE5F50E74283 IInteractionTrackerInertiaNaturalMotionStatics
 CFDBB9B0-0CA0-11D2-AA68-00C04FA35B82 ISmtpOutCommandSink
@@ -12301,6 +12525,7 @@ D1D4A705-EB0B-4ECB-B9B7-E65D03087C8E IRichTextBlockOverflowStatics3
 D1DB340F-1BC4-4CA8-BCF7-770BFF9B27AB ITextElement3
 D1E7AFEB-6A2E-11D0-8C78-00C04FD918B4 IMenuPopup
 D1EB4C5C-1B53-4E4A-8B5C-DB7887AC949B IVariablePhotoCapturedEventArgs
+D1EB6D20-8923-11D0-9D97-00A0C90A43CB DirectPlay
 D1EBC1D6-8B60-4C20-9CAF-E59382E7C400 IDvbDataBroadcastDescriptor
 D1F0C730-DA40-4328-BF76-5156FA33B747 IPosPrinterStatus
 D1F87ADE-ECA1-4561-A32B-64601B4E0597 IUIElementStatics3
@@ -12348,6 +12573,58 @@ D27C3481-5A1C-45B2-8AAA-C20EBBE8229E ICredentialProvider
 D2876FC4-D410-4EB5-9E69-C9AA1277EAF0 IEqualizerEffectDefinitionFactory
 D2959BF7-B31B-4A3D-9600-712EB1335BA4 IPrintDocumentPackageTargetFactory
 D2AC1B6A-D29F-554B-B303-F0452345FE02 IClipboardStatics2
+D2AC2878-B39B-11D1-8704-00600893B1BD IDirectMusicBuffer
+D2AC287A-B39B-11D1-8704-00600893B1BD IDirectMusicPortDownload
+D2AC287B-B39B-11D1-8704-00600893B1BD IDirectMusicDownload
+D2AC287C-B39B-11D1-8704-00600893B1BD IDirectMusicCollection
+D2AC287D-B39B-11D1-8704-00600893B1BD IDirectMusicInstrument
+D2AC287E-B39B-11D1-8704-00600893B1BD IDirectMusicDownloadedInstrument
+D2AC2881-B39B-11D1-8704-00600893B1BD DirectMusicPerformance
+D2AC2882-B39B-11D1-8704-00600893B1BD DirectMusicSegment
+D2AC2883-B39B-11D1-8704-00600893B1BD DirectMusicSegmentState
+D2AC2884-B39B-11D1-8704-00600893B1BD DirectMusicGraph
+D2AC2885-B39B-11D1-8704-00600893B1BD DirectMusicTempoTrack
+D2AC2886-B39B-11D1-8704-00600893B1BD DirectMusicSeqTrack
+D2AC2887-B39B-11D1-8704-00600893B1BD DirectMusicSysExTrack
+D2AC2888-B39B-11D1-8704-00600893B1BD DirectMusicTimeSigTrack
+D2AC288A-B39B-11D1-8704-00600893B1BD DirectMusicStyle
+D2AC288B-B39B-11D1-8704-00600893B1BD DirectMusicChordTrack
+D2AC288C-B39B-11D1-8704-00600893B1BD DirectMusicCommandTrack
+D2AC288D-B39B-11D1-8704-00600893B1BD DirectMusicStyleTrack
+D2AC288E-B39B-11D1-8704-00600893B1BD DirectMusicMotifTrack
+D2AC288F-B39B-11D1-8704-00600893B1BD DirectMusicChordMap
+D2AC2890-B39B-11D1-8704-00600893B1BD DirectMusicComposer
+D2AC2892-B39B-11D1-8704-00600893B1BD DirectMusicLoader
+D2AC2894-B39B-11D1-8704-00600893B1BD DirectMusicBandTrack
+D2AC2896-B39B-11D1-8704-00600893B1BD DirectMusicChordMapTrack
+D2AC2898-B39B-11D1-8704-00600893B1BD DirectMusicMuteTrack
+D2AC2899-B39B-11D1-8704-00600893B1BD NOTIFICATION_SEGMENT
+D2AC289A-B39B-11D1-8704-00600893B1BD NOTIFICATION_MEASUREANDBEAT
+D2AC289B-B39B-11D1-8704-00600893B1BD NOTIFICATION_CHORD
+D2AC289C-B39B-11D1-8704-00600893B1BD NOTIFICATION_COMMAND
+D2AC289D-B39B-11D1-8704-00600893B1BD CommandParam
+D2AC289E-B39B-11D1-8704-00600893B1BD ChordParam
+D2AC289F-B39B-11D1-8704-00600893B1BD RhythmParam
+D2AC28A1-B39B-11D1-8704-00600893B1BD IDirectMusicStyle
+D2AC28A4-B39B-11D1-8704-00600893B1BD TimeSignature
+D2AC28A5-B39B-11D1-8704-00600893B1BD TempoParam
+D2AC28A7-B39B-11D1-8704-00600893B1BD Download
+D2AC28A8-B39B-11D1-8704-00600893B1BD Unload
+D2AC28A9-B39B-11D1-8704-00600893B1BD Enable_Auto_Download
+D2AC28AA-B39B-11D1-8704-00600893B1BD Disable_Auto_Download
+D2AC28AB-B39B-11D1-8704-00600893B1BD Clear_All_Bands
+D2AC28AC-B39B-11D1-8704-00600893B1BD IDirectMusicBand
+D2AC28AD-B39B-11D1-8704-00600893B1BD IDirectMusicChordMap
+D2AC28AF-B39B-11D1-8704-00600893B1BD MuteParam
+D2AC28B0-B39B-11D1-8704-00600893B1BD PerfMasterTempo
+D2AC28B1-B39B-11D1-8704-00600893B1BD PerfMasterVolume
+D2AC28B2-B39B-11D1-8704-00600893B1BD PerfMasterGrooveLevel
+D2AC28B5-B39B-11D1-8704-00600893B1BD IDirectMusicObject
+D2AC28BA-B39B-11D1-8704-00600893B1BD IDirectMusicTool
+D2AC28BD-B39B-11D1-8704-00600893B1BD IDirectMusicStyle
+D2AC28BE-B39B-11D1-8704-00600893B1BD IDirectMusicChordMap
+D2AC28BF-B39B-11D1-8704-00600893B1BD IDirectMusicComposer
+D2AC28C0-B39B-11D1-8704-00600893B1BD IDirectMusicBand
 D2AE94F5-3D15-4F79-9C0C-EAAFA675155C IGattServiceUuidsStatics2
 D2B57227-3D23-4B95-93C0-492BD454C356 IInsertItem
 D2BC4C84-3F72-4A52-A604-7BCBF3982CBB INewWindowManager
@@ -12378,6 +12655,7 @@ D3609541-1B29-4DE5-A4AD-5AEBAF319512 IRTCMessagingEvent
 D36D6542-9085-438E-97BA-A25C976BE2FD ICurrentAppStaticsWithFiltering
 D37771A8-DDAD-4288-8428-D1CAE394128B IDataPackageView3
 D384759F-34F6-4482-8435-F552F9B24CC8 IWindow2
+D38894D1-C052-11D2-872F-00600893B1BD IDirectMusicSegment2
 D396EC8C-605E-4249-978D-72AD1C312872 IMFSpatialAudioObjectBuffer
 D3972AF8-40C0-4FD7-AA6C-0AA13CA6188C IAppInstallItem2
 D397E42E-1488-42F2-BF64-253F4884BFED IContactReader
@@ -12389,6 +12667,8 @@ D3BE3DFB-9F54-4642-A7A5-5C18D5EE2A3F ITextRangeProvider2
 D3CA3D42-09B1-49D5-8891-E7B5648E02A2 ISetterFactory
 D3CFB977-0084-41D7-A221-28158D3BC32C IAutomationPeer3
 D3E07363-087C-476C-86A7-DBB15F46DDB4 IWTSPluginServiceProvider
+D3E16E80-7835-11CF-8F52-0040333594A3 D3DRMMaterialEmissiveColor
+D3E16E81-7835-11CF-8F52-0040333594A3 D3DRMColorRGB
 D3E18888-F120-4FEE-8C68-35296EAE91D4 IXpsOMPage
 D3E1A2AA-CAAC-460E-B98A-47F9F318A1FA WdsTransportNamespaceScheduledCastManualStart
 D3E8D338-FDC4-4AFC-80B5-D532A1782EE1 IXpsSignatureManager
@@ -12498,6 +12778,7 @@ D5DF3F03-4BDE-4417-AEFE-71177BDAEA66 IRTCClientPortManagement
 D5E62A0B-696D-462F-94DF-72E33CEF2659 IOpcDigitalSignatureManager
 D5E8041D-920F-45E9-B8FB-B1DEB82C6E5E InternetExplorerMedium
 D5F1726B-BDCE-11DC-A8A8-001321F1405F MBN_REGISTRATION_CONSTANTS
+D5F1CAE0-4BD7-11D1-B974-0060083E45F3 IDirect3DRMAnimationArray
 D5F569D0-593B-101A-B569-08002B2DBF7A IPSFactoryBuffer
 D5F56A34-593B-101A-B569-08002B2DBF7A IRpcProxyBuffer
 D5F56AFC-593B-101A-B569-08002B2DBF7A IRpcStubBuffer
@@ -12585,6 +12866,7 @@ D7A64502-DDDA-55CD-B4C9-E79ABC8B0EA4 IIconSourceElement
 D7AB3341-C9D3-11D1-BB47-0080C7C5A2C0 IMSMQPrivateEvent
 D7AD183E-38F5-4210-B55F-EC8D601BBD47 IIsdbTSInformationDescriptor
 D7B13A01-F7D4-42A6-8595-12FC8C24E851 RDPSRAPIAttendeeManager
+D7B70EE0-4340-11CF-B063-0020AFC2CD35 DirectDraw
 D7C59F46-8DA8-4F70-9751-E53BB6DF4596 ICoreIncrementalInkStrokeFactory
 D7CDA5EB-CBD7-4F3B-8526-B506DEC35C53 IRcsEndUserMessage
 D7D17254-A4FD-55C4-98CF-F2710B7D8BE2 IFindRelatedPackagesOptionsFactory
@@ -13061,6 +13343,7 @@ E0777CFD-F7E4-4A67-9AC0-A994FCACD020 IListViewItemPresenterFactory
 E07FA734-A0B6-5755-9E87-24F54CCA9372 IExecuteRequestedEventArgs
 E08BB4BD-5D30-4B5A-92CC-D54DF881D49E IMobileBroadbandDeviceInformation3
 E08DEEF9-5363-406E-9FDC-080EE247BBE0 ISpatialAudioObjectRenderStreamForHrtf
+E09C77E0-DA60-11CF-9B06-00A0C903A3B8 DDVPTYPE_E_HREFL_VREFL
 E0A9070D-4C42-4A90-983A-75F5A83A2FBE IDiscretePointKeyFrame
 E0AFDE10-55CF-4C01-91D4-A20557D849F0 IHttpDiagnosticProviderRequestResponseTimestamps
 E0B545F8-3B0F-44DA-924C-7B18BAAA62A9 IWebAccountProviderSilentReportOperation
@@ -13156,6 +13439,10 @@ E2AE82B7-229D-4428-B3D2-500000000009 _IMAPI_FORMAT2_RAW_CD_WRITE_ACTION
 E2B3C97F-6AE1-41AC-817A-F6F92166D7DD NetFwPolicy2
 E2B50029-B4C1-4314-A4B8-FB813A2F275E IDataReader
 E2B7A27B-4D40-471D-92ED-63375B18C788 IClaimedPosPrinterClosedEventArgs
+E2BF56C0-840F-11CF-8F52-0040333594A3 D3DRMAnimationOptions
+E2BF56C1-840F-11CF-8F52-0040333594A3 D3DRMFramePosition
+E2BF56C2-840F-11CF-8F52-0040333594A3 D3DRMFrameVelocity
+E2BF56C3-840F-11CF-8F52-0040333594A3 D3DRMFrameRotation
 E2CC1366-CF66-43C4-A96A-A5A112DB4746 IContactPickerUI
 E2CD3A4B-AF25-4D3D-9110-DA0E6F8EE877 IMFSourceBuffer
 E2CD4A63-2B72-4D48-B739-95E4765195BA IAccStore
@@ -13174,6 +13461,7 @@ E318AD57-0AA0-450F-ACA5-6FAB7103D917 IPersistSerializedPropStorage
 E31FB81B-1335-11D1-8189-0000F87557DB IDXEffect
 E32ABF70-D8FA-45EC-A942-D2E26FB60BA5 ISystemDataPaths
 E32BDE1C-28AF-4B90-95DA-A3ACF320B929 IPrintCustomOptionDetails
+E3357330-CC5E-11D2-A434-00A0C90629A8 ID3DXMatrixStack
 E33574EA-1076-44D1-9383-DC24AC5CFF2A IPickerFlyoutBase
 E341516B-2E32-11D1-9964-00C04FBBB345 IEventPublisher
 E348A69A-8889-4CA3-BE07-D5FFFF5F0804 IAppDiagnosticInfo
@@ -13225,6 +13513,7 @@ E433A2D7-AAB7-4749-A66C-1530941AC47B IResourceStringView
 E44327C3-A999-485B-B1D8-72DC517821E6 IPrintDocument
 E44C3566-915D-4070-99C6-047BFF5A08F5 ILegacyIAccessibleProvider
 E44E9428-BDBC-4987-A099-40DC8FD255E7 ExecuteUnknown
+E4524541-8EA5-11D1-8A96-006097B01411 DPAID_INetPort
 E454B549-4B2C-42AD-B04B-D35947D1EE50 IRangeBaseAutomationPeer
 E46135BD-0FA2-416C-91B4-C1E48CA061C1 IBluetoothClassOfDeviceStatics
 E4676FFD-2D92-4C08-BAD4-F6590B54A6D3 IWorkplaceSettingsStatics
@@ -13279,6 +13568,7 @@ E56E4CBD-8F70-49D8-A0F8-EDB3D6AB9BF2 IMFTimer
 E56F9F8F-EE99-4DCA-A3C3-70AF3DB44F5F IAppBroadcastingUI
 E572D3C9-37BE-4AE2-825D-D521763E3108 IShellNameSpace
 E5732044-F49F-4B60-8DD4-5E7E3A632AC0 IDisplayRequest
+E5745280-B24F-11CF-9DD5-00AA00A71A2F D3DRMAppData
 E5791FAE-5665-4E0C-95BE-5FDE31644185 IRadioInstanceCollection
 E579AB5F-1CC4-44B4-BED9-DE0991FF0623 VSSCoordinator
 E57AE83C-2216-4717-9EF6-4662334C6501 ISwipeControlStatics
@@ -13314,6 +13604,7 @@ E6257174-A060-4C9A-A088-3B1B471CAD28 IMFContentProtectionDevice
 E62BA82F-A63A-430B-BF62-29E93E5633B4 IStreamSocketListenerInformation
 E62BD252-C566-4F42-9974-916C4D76377E IUserActivityAttributionFactory
 E62C9A52-AC94-5750-AC1B-6CFB6F250263 IAppUriHandlerRegistrationManager
+E63232A0-9DBF-11D0-9CC1-00A0C905425E DPAID_INetW
 E634F691-525A-4CE2-8FCE-AA4162579154 IMobileBroadbandUicc
 E63DE750-3BD7-4BE5-9C84-6B4281988C44 CSearchScopeRule
 E6442437-6C68-4F52-94DD-2CFED267EFB9 TrayDeskBand
@@ -13408,6 +13699,7 @@ E7D7AD42-BD3D-11D1-9A4D-00C04FC297EB ICertManageModule
 E7D83A5A-DDFA-5DE0-8B70-C842C21988EC IFileSavePicker4
 E7D958D7-4D13-4237-A365-4C4F6A4CFD2F ICastingDeviceStatics
 E7DBF5A8-1DB9-4783-876B-3DFE12ACFDB3 IMediaEncodingProfile
+E7E1F7C0-88D2-11D0-9AD0-00A0C9A06E35 IDirectInputEffect
 E7E26076-A044-48E2-A512-3094D574C7CC IAppRecordingManager
 E7EA222F-DEE6-4393-A8B2-8923D641F395 ITriggerBase
 E7ED1C71-87F7-4378-A840-C9200DACEE47 ISpellChecker2
@@ -13509,6 +13801,7 @@ E9CD0383-A268-4BB4-82DE-658D53574D41 IMFNetProxyLocator
 E9CD0384-A268-4BB4-82DE-658D53574D41 IMFNetProxyLocatorFactory
 E9D3BE02-BFDF-437F-8C29-88C77393D309 IHingeAngleSensor
 E9DB4E34-E510-44B7-809D-F2F85B555339 IProviderI2cConnectionSettings
+E9E19280-6E05-11CF-AC4A-0000C03825A1 IDirect3DRMDevice
 E9E9D07D-6340-4828-AD12-690694B9910B IPointerUpThemeAnimation
 E9E9D8A3-4EDD-11D0-874F-00AA00530EE9 IIMAPCallback
 E9E9D8A8-4EDD-11D0-874F-00AA00530EE9 IIMAPTransport
@@ -13578,6 +13871,24 @@ EB0DBE66-7448-448D-9DB8-D78ACD2ABBA9 IAdaptiveNotificationContent
 EB0FE172-1A3A-11D0-89B3-00A0C90A90AC IDeskBand
 EB0FE173-1A3A-11D0-89B3-00A0C90A90AC IDeskBar
 EB15BC42-C0A9-46C6-AC24-B83DE429C733 ICheckBoxAutomationPeer
+EB16CB00-D271-11CE-AC48-0000C03825A1 IDirect3DRMObject
+EB16CB02-D271-11CE-AC48-0000C03825A1 IDirect3DRMViewport
+EB16CB03-D271-11CE-AC48-0000C03825A1 IDirect3DRMFrame
+EB16CB04-D271-11CE-AC48-0000C03825A1 IDirect3DRMVisual
+EB16CB07-D271-11CE-AC48-0000C03825A1 IDirect3DRMFace
+EB16CB08-D271-11CE-AC48-0000C03825A1 IDirect3DRMLight
+EB16CB09-D271-11CE-AC48-0000C03825A1 IDirect3DRMTexture
+EB16CB0A-D271-11CE-AC48-0000C03825A1 IDirect3DRMWrap
+EB16CB0B-D271-11CE-AC48-0000C03825A1 IDirect3DRMMaterial
+EB16CB0D-D271-11CE-AC48-0000C03825A1 IDirect3DRMAnimation
+EB16CB0E-D271-11CE-AC48-0000C03825A1 IDirect3DRMAnimationSet
+EB16CB10-D271-11CE-AC48-0000C03825A1 IDirect3DRMDeviceArray
+EB16CB11-D271-11CE-AC48-0000C03825A1 IDirect3DRMViewportArray
+EB16CB12-D271-11CE-AC48-0000C03825A1 IDirect3DRMFrameArray
+EB16CB13-D271-11CE-AC48-0000C03825A1 IDirect3DRMVisualArray
+EB16CB14-D271-11CE-AC48-0000C03825A1 IDirect3DRMLightArray
+EB16CB16-D271-11CE-AC48-0000C03825A1 IDirect3DRMPickedArray
+EB16CB17-D271-11CE-AC48-0000C03825A1 IDirect3DRMFaceArray
 EB1F8C5B-0ABB-4E70-B8A8-620D0D953AB2 ICacheModeFactory
 EB23D22B-A685-42AE-8495-B3119ADB4167 IAppointmentCalendar3
 EB2AD723-E5E2-4AE2-A7A7-A1097A098E90 IRemoteSystem5
@@ -13655,6 +13966,7 @@ ED02D274-81F1-5AC2-A0C3-BBDAF5A3FFEE IGrid4
 ED1343B0-A8A6-11D0-A9EA-00AA00685C74 CSEOStream
 ED17C908-A695-4CF6-92FD-97263BA71047 ISpatialAnchor2
 ED1A54B8-5EFA-482A-93C0-8AD86F0D68C3 GPMBackup
+ED1EC5C0-C0A8-11D0-941C-0080C80CFA7B D3DRMMeshFaceWraps
 ED20DC5C-D21B-463E-A9B7-C1FD0EDAE978 IWebAccountProviderManageAccountOperation
 ED2507B8-2475-439C-85FF-7FED341FDC88 ISpatialAnchorExporterStatics
 ED25831F-90DB-498D-A7B4-EBCE807D3C23 IESimProfileMetadata
@@ -13744,6 +14056,7 @@ EEB14B68-808B-4ABE-A5EA-B51DA7588008 ISpeechMemoryStream
 EEB20886-E470-4CF6-842B-2739C0EC5CFB IRDPSRAPISharingSession
 EEBF1F5B-07C1-4447-A3AB-22ACAF78A804 IWICMetadataReaderInfo
 EEC3CA84-9896-4A7D-BB35-6FB21EAECA11 IDatePickerFactory
+EEC6993A-B3FD-11D2-A916-00C04FB98638 IDirectInputPIDDriver
 EECD2C1C-B0D0-42E7-B137-B89B16244D41 IPosPrinterStatics2
 EED9C2EE-66B4-4F18-A697-AC7D3960215C IMFSensorTransformFactory
 EEDD23E0-8410-11CE-A1C3-08002B2B8D8F IAccessControl
@@ -13873,6 +14186,7 @@ F12EF0B5-BEFB-46C9-9EE5-8A0681F13F8C IUserDataPathsInterop
 F1346DF2-C282-4E22-B918-743A929A8D55 IAppxFactory2
 F134C4B7-B1F8-4E75-B886-74B90943BECB IWinInetFileStream
 F141C0D6-4BFF-4F23-BA66-6045B137D5DF IAsymmetricAlgorithmNamesStatics2
+F14599E5-4689-11D2-AFA6-00AA0024D8B6 DMUS_PROP_DLS2
 F156A638-7997-48D9-A10C-269D46533F48 IDeviceInformation2
 F15C60F7-D5E8-5E92-556D-E03B60253104 IChatMessageManagerStatic
 F164EDF1-CC7C-4F0D-9A94-34222625C393 IInternetSecurityManagerEx
@@ -13881,6 +14195,8 @@ F16AABCF-7F59-4A51-8494-BD89D5FF50EE IESimServiceInfo
 F16E3162-8BB8-11DF-A3F3-A420E0D72085 IRdRoleRdcb
 F174E5B0-9046-11D0-A9E8-00AA00685C74 IMCISMessageFilter
 F179F272-9846-4632-8B9C-BE6FA8D3C9BB IRatingControlAutomationPeerFactory
+F17E8672-C3B4-11D1-870B-00600893B1BD DirectMusicSignPostTrack
+F17E8673-C3B4-11D1-870B-00600893B1BD DefaultGMCollection
 F17F9DB1-74CD-4787-9787-19866E9A5559 ICurrentAppSimulator
 F185FE76-E64E-11D2-B76E-00C04FB6BD3D IAMPushSource
 F19104BE-422A-4904-A52F-EE72010429E5 IFrameworkElement2
@@ -14030,6 +14346,7 @@ F2E6072D-2631-11D1-89F1-00A0C90D061E ClusResDependents
 F2E6072E-2631-11D1-89F1-00A0C90D061E ISClusResDependents
 F2E771ED-B803-5137-9435-1C96E48721E9 IInteractionTrackerIdleStateEnteredArgs2
 F2F00BC2-BD2B-4A01-A8FB-323E0158527C ISpatialInteractionSourceState3
+F2F0CE00-E0AF-11CF-9C4E-00A0C905425E DPAID_ComPort
 F2F44344-7372-5F7B-9B34-29C944F5A715 IBluetoothLEPreferredConnectionParameters
 F2F8CBB3-B040-11D0-BB16-00C04FB66F63 IUrlTrackingStg
 F303043A-E8BF-4E8E-AE69-C9DADD57A114 ICoreClosestInteractiveBoundsRequested
@@ -14084,6 +14401,7 @@ F3F1F444-96AE-4DE0-B540-2B3246E6968C IAdaptiveMediaSourceDownloadBitrateChangedE
 F3F44350-581E-4ECC-9F59-E507D04F06E6 IHttpCookiePairHeaderValueCollection
 F3F4868F-29D4-4094-8C54-EA61A88294A4 IGridViewItemDataAutomationPeer
 F404E97B-9034-453C-8EBF-140A38C86F1D IContactName
+F406B180-7B3B-11CF-8F52-0040333594A3 D3DRMTimedFloatKeys
 F4104FCE-9A70-11D0-8FDE-00C04FD9189D IDirectDrawMediaStream
 F4104FCF-9A70-11D0-8FDE-00C04FD9189D IDirectDrawStreamSample
 F411D4FD-14BE-4260-8C40-03B7C95E608A IFsrmSetting
@@ -14161,6 +14479,7 @@ F53ACB06-CF6A-4387-A3FE-5221F3E7E0E0 INaturalMotionAnimationFactory
 F53BC5AC-84CA-4972-8CE9-E58F97B37A50 ISmartCardTrigger
 F546656F-E106-4C82-A90F-554012904B85 IRacingWheel
 F55AB64A-5504-5D9E-9D20-2FFDB88FC56B ICalendarView2
+F56920A0-D218-11D0-BA39-00C04FD7ED67 DPLPROPERTY_LobbyGuid
 F5694708-88FE-4B35-BABF-E56162D5FBC8 GPM
 F56A206E-1B77-42EF-8F0D-B949F8D9B260 IPlayToManager
 F56D926A-55ED-49C1-B09C-4B8223B50DB3 IGuidanceRouteStatics
@@ -14238,11 +14557,18 @@ F6D90F12-9C73-11D3-B32E-00C04F990BB4 FreeThreadedDOMDocument
 F6D90F14-9C73-11D3-B32E-00C04F990BB4 DSOControl
 F6D90F16-9C73-11D3-B32E-00C04F990BB4 XMLHTTP
 F6DC31E7-4B13-45C5-8897-7D71269E085D ISelectableWordsSegmenter
+F6DCC200-A2FE-11D0-9C4F-00A0C905425E DPAID_Modem
 F6E022BA-BFF3-42EC-9033-E073F33E83C3 IUIAnimationManagerEventHandler2
 F6E4C0A0-46D9-4FB8-BE21-57A3EF2B626C IAudioClockAdjustment
 F6EBD84D-37E8-46EE-A574-41AD550482E0 IFontIcon2
 F6ED581A-49A5-47E2-B771-FD8DC02B6259 GPMGPOLinksCollection
 F6EF6140-E26F-4D82-BAC4-E9BA5FD239A8 ICreateProcessInputs
+F6F23F40-7686-11CF-8F52-0040333594A3 D3DRMMeshTextureCoords
+F6F23F41-7686-11CF-8F52-0040333594A3 D3DRMFrameTransformMatrix
+F6F23F42-7686-11CF-8F52-0040333594A3 D3DRMMeshMaterialList
+F6F23F43-7686-11CF-8F52-0040333594A3 D3DRMMeshNormals
+F6F23F44-7686-11CF-8F52-0040333594A3 D3DRMCoords2d
+F6F23F45-7686-11CF-8F52-0040333594A3 D3DRMMatrix4x4
 F6F83553-AA27-4731-8E4A-3DEC5CE9EEC9 IContactConnectedServiceAccount
 F6F8F5CE-209A-48FB-86FC-D44578F317E6 IMediaDeviceController
 F6FC2A9B-D5BC-4241-B436-1B8460C13832 IRTCSessionOperationCompleteEvent2
@@ -14384,6 +14710,8 @@ F9581718-2283-4336-9397-1C6C067299D2 ICallWaitingSettingsCompletion
 F95B9AAC-A621-488E-9156-8EE31165BE04 IItemsPickedEventArgs
 F95C7E80-BD63-4601-9782-445EBFF011FC IAnnotationProvider
 F95D89FB-8AF0-4CB0-926D-696866E5046A IDepthCorrelatedCoordinateMapper
+F96029A1-4282-11D2-8717-00600893B1BD IDirectMusicTrack
+F96029A2-4282-11D2-8717-00600893B1BD IDirectMusicSegment
 F96C2753-E1D4-5142-86C2-7318BFBBC705 IPasswordBox5
 F9769A06-7ACA-4E39-9CFB-97BB35F0E77E AuxInTuningSpace
 F97933B0-8DAE-4496-8997-5BA015FE0D82 IUIAutomationActiveTextPositionChangedEventHandler
@@ -14461,6 +14789,7 @@ FAF73DB4-4CFF-44C7-8445-040E6EAD0063 IGattLocalCharacteristicParameters
 FAFEC376-F22E-466D-913C-AE24CCDB160F IGridViewItemAutomationPeerFactory
 FB07B732-584D-4583-9C60-641FEA274349 IMapRoute
 FB07D14C-3380-457A-A987-991357896C1B ICoreWetStrokeUpdateEventArgs
+FB09565B-3631-11D2-BCB8-00A0C922E6EB PerfAutoDownload
 FB0EE829-EC0C-449A-83DA-7AE3073365A2 IPaymentMediator
 FB0FADBA-C7F0-4BB6-9F6B-BA5D73209ACA IPhoneCallHistoryEntryAddressFactory
 FB1232D6-EE73-46E7-8761-11CD0157728F IContactLaunchActionVerbsStatics
@@ -14550,6 +14879,7 @@ FC8946BD-A3A2-4969-8174-25B5D3C28033 IListViewItemPresenter
 FC8CCBBF-5549-4B87-959F-23CA401FD473 IAppBroadcastBackgroundService2
 FC967639-88E6-4646-9112-4344AAEC9DFA ISpatialInteraction
 FC9AC5CC-C1D7-4153-923B-A2D43C6C8D49 ISmartCardCryptogramMaterialCharacteristics
+FCA326A0-DA60-11CF-9B06-00A0C903A3B8 DDVPTYPE_CCIR656
 FCA6C349-A12F-43A3-8DD6-5A5A4282577B ITfFnCustomSpeechCommand
 FCAA3781-3083-49AB-BE15-56DFBBB74D6F ITextConversionGeneratorFactory
 FCADBFDF-DC27-5638-B7F2-EBDFA4F7EA93 IDisplayStateOperationResult
@@ -14687,6 +15017,7 @@ FECEAAA5-8405-11CF-8BA1-00AA00476DA6 IOmNavigator
 FECEAAA6-8405-11CF-8BA1-00AA00476DA6 HTMLNavigator
 FECEAFFD-C441-11D1-960E-00C04FBD7C09 IMimeObjResolver
 FEDDFCD4-FA12-4435-AE55-7AD1A9779AF7 ITSGAuthorizeResourceSink
+FEDFAE25-E46E-11D1-AACE-0000F875AC12 DMUS_PROP_Volume
 FEE0EF8B-46BD-4DB4-B7E6-FF2C687313BC ISyncMgrEvent
 FEE200FB-8664-44E0-95D9-42696199BFFC IHttpResponseMessage
 FEE25B97-8B96-4602-81D2-82FD8E0F7EA8 IListPickerFlyoutStatics
@@ -14717,6 +15048,11 @@ FF513437-DF9F-4DF0-BF82-0EC5D7B35AAE IStreamSocketListener
 FF5693BE-2CE0-4D48-B5C5-40817D1ACDB9 IShellItemResources
 FF6123CF-4386-4AA3-B73D-B804E5B63B23 IUserNotificationListenerStatics
 FF68468D-CA92-4434-9743-0E90988AD879 IGattServiceProviderAdvertisingParameters2
+FF6B7F70-A40E-11D1-91F9-0000F8758E66 IDirect3DRMFrame3
+FF6B7F73-A40E-11D1-91F9-0000F8758E66 IDirect3DRMTexture3
+FF6B7F75-A40E-11D1-91F9-0000F8758E66 IDirect3DRMMaterial2
+FF6B7F77-A40E-11D1-91F9-0000F8758E66 IDirect3DRMAnimation2
+FF6B7F79-A40E-11D1-91F9-0000F8758E66 IDirect3DRMAnimationSet2
 FF8BE292-AF28-47EB-A5C3-DC44CED40FD2 IContentPresenterStatics4
 FF905D4C-F83B-41B0-8A0C-1462D9C56B73 IConnectionSession
 FF99081E-10C4-44B7-9FEA-A748B5A46F31 ISpiConnectionSettingsFactory
-- 
2.43.0

